roachtest: activerecord failed

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5840461?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5840461?buildTab=artifacts#/activerecord) on master @ [8fd5b3500796fae41c07fffd4246648b349b6460](https://github.com/cockroachdb/cockroach/commits/8fd5b3500796fae41c07fffd4246648b349b6460):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1833-g8fd5b35007
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7363 tests passed
		9 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		9 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: BasicsTest#test_default_values_are_deeply_dupped (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #84954 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-21.1 release-blocker]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-17969

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5846431?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5846431?buildTab=artifacts#/activerecord) on master @ [eabdc49383a19d5731b765dbb0d8b45bd9e24404](https://github.com/cockroachdb/cockroach/commits/eabdc49383a19d5731b765dbb0d8b45bd9e24404):


```
		--- FAIL: DefaultTextTest#test_default_texts (unexpected)
		--- FAIL: RelationTest#test_automatically_added_where_references (unexpected)
		--- FAIL: BasicsTest#test_previously_changed_dup (unexpected)
		--- FAIL: BasicsTest#test_when_ignored_attribute_is_loaded,_cast_type_should_be_preferred_over_DB_type (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping (unexpected)
		--- FAIL: BasicsTest#test_find_ordered_last (unexpected)
		--- FAIL: CalculationsTest#test_should_group_by_association_with_non_numeric_foreign_key (unexpected)
		--- FAIL: PersistenceTest#test_create_many (unexpected)
		--- FAIL: HasManyAssociationsTest#test_does_not_duplicate_associations_when_used_with_natural_primary_keys (unexpected)
		--- FAIL: BasicsTest#test_dont_clear_sequence_name_when_setting_explicitly (unexpected)
		--- FAIL: PersistenceTest#test_class_level_update_without_ids (unexpected)
		--- FAIL: BasicsTest#test_preserving_time_objects_with_time_with_zone_conversion_to_default_timezone_utc (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_nullify_foreign_key (unexpected)
		--- FAIL: HasManyThroughDisableJoinsAssociationsTest#test_counting_on_disable_joins_through_using_custom_foreign_key (unexpected)
		--- FAIL: HasManyThroughFixture#test_has_many_through_with_default_table_name_on_join_table (unexpected)
		--- FAIL: CalculationsTest#test_offset_is_kept (unexpected)
		--- FAIL: BasicsTest#test_attributes_on_dummy_time (unexpected)
		--- FAIL: PersistenceTest#test_update_columns_returns_boolean (unexpected)
		--- FAIL: PersistenceTest#test_delete_new_record (unexpected)
		--- FAIL: NestedThroughAssociationsTest#test_nested_has_many_through_with_conditions_on_through_associations_preload (unexpected)
		--- FAIL: BasicsTest#test_benchmark_with_log_level (unexpected)
		--- FAIL: RelationTest#test_reverse_arel_assoc_order_with_function (unexpected)
		--- FAIL: NestedThroughAssociationsTest#test_has_many_through_belongs_to_with_has_many_through_source_reflection_preload (unexpected)
		--- FAIL: BasicsTest#test_scoped_can_take_a_values_hash (unexpected)
		--- FAIL: CalculationsTest#test_pluck_and_distinct (unexpected)
		--- FAIL: RelationTest#test_default_reverse_order_on_table_without_primary_key (unexpected)
		--- FAIL: PersistenceTest#test_persisted_returns_boolean (unexpected)
		--- FAIL: BasicsTest#test_distinct_delegates_to_scoped (unexpected)
		--- FAIL: RelationTest#test_eager_load_with_blank_value (unexpected)
		--- FAIL: HasOneThroughAssociationsTest#test_has_one_through_with_has_one (unexpected)
		--- FAIL: NestedThroughAssociationsTest#test_nested_has_many_through_with_a_table_referenced_multiple_times (unexpected)
		--- FAIL: RelationTest#test_loaded_relations_cannot_be_mutated_by_multi_value_methods (unexpected)
		--- FAIL: RelationTest#test_#skip_query_cache!_with_an_eager_load (unexpected)
		--- FAIL: CalculationsTest#test_count_selected_arel_attribute (unexpected)
		--- FAIL: BasicsTest#test_clone_of_new_object_with_defaults (unexpected)
		--- FAIL: RelationTest#test_find_by_with_non-hash_conditions_returns_the_first_matching_record (unexpected)
		--- FAIL: CalculationsTest#test_should_count_with_manual_distinct_select_and_distinct (unexpected)
		--- FAIL: PersistenceTest#test_update_attribute_in_before_validation_respects_callback_chain (unexpected)
		--- FAIL: RelationTest#test_relation_join_method (unexpected)
		--- FAIL: CalculationsTest#test_plucks_with_ids (unexpected)
		--- FAIL: PrimaryKeysTest#test_read_attribute_with_custom_primary_key (unexpected)
		--- FAIL: HasManyThroughDisableJoinsAssociationsTest#test_polymophic_disable_joins_through_counting (unexpected)
		--- FAIL: RelationTest#test_create_bang (unexpected)
		--- FAIL: CalculationsTest#test_should_calculate_grouped_association_with_foreign_key_option (unexpected)
		--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_1#test_create_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
		--- FAIL: PrimaryKeysTest#test_update_with_non_primary_key_id_column (unexpected)
		--- FAIL: BasicsTest#test_clone_preserves_subtype (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #84954 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-21.1 release-blocker]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5851979?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5851979?buildTab=artifacts#/activerecord) on master @ [eabdc49383a19d5731b765dbb0d8b45bd9e24404](https://github.com/cockroachdb/cockroach/commits/eabdc49383a19d5731b765dbb0d8b45bd9e24404):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1838-geabdc49383
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #84954 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-21.1 release-blocker]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5864984?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5864984?buildTab=artifacts#/activerecord) on master @ [8db26695114981eec5c12317f3dc5d770b522a3f](https://github.com/cockroachdb/cockroach/commits/8db26695114981eec5c12317f3dc5d770b522a3f):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1871-g8db2669511
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5875958?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5875958?buildTab=artifacts#/activerecord) on master @ [0f100f3a09246bc80ef2c3119e4ec1c24ef36801](https://github.com/cockroachdb/cockroach/commits/0f100f3a09246bc80ef2c3119e4ec1c24ef36801):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1898-g0f100f3a09
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7358 tests passed
		14 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		14 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_change_column_of_parent_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_rename_reference_column_of_child_table (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_foreign_key_by_column (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5889604?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5889604?buildTab=artifacts#/activerecord) on master @ [7c95ccd0a9f0b0aca9cdb0ed813caf8d60b28732](https://github.com/cockroachdb/cockroach/commits/7c95ccd0a9f0b0aca9cdb0ed813caf8d60b28732):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1939-g7c95ccd0a9
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7358 tests passed
		14 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		14 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_reference_column_of_child_table (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_change_column_of_parent_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5902767?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5902767?buildTab=artifacts#/activerecord) on master @ [1129fbc650fe3a037b03aea1e5f1d8078618cb1c](https://github.com/cockroachdb/cockroach/commits/1129fbc650fe3a037b03aea1e5f1d8078618cb1c):


```
		--- FAIL: ActiveRecord::ConnectionAdapters::LegacyConnectionHandlersMultiDbTest#test_loading_relations_with_multi_db_connection_handlers (unexpected)
		--- FAIL: CockroachDB::PostgresqlBigSerialTest#test_not_bigserial_column (unexpected)
		--- FAIL: PersistenceTest#test_update_many_with_array_of_active_record_base_objects! (unexpected)
		--- FAIL: BasicsTest#test_many_mutations (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_with_hash_conditions (unexpected)
		--- FAIL: ReadOnlyTest#test_cant_update_column_readonly_record (unexpected)
		--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociation#test_should_not_update_children_when_parent_creation_with_no_reason (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_unavailable_through_reflection (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_find_conditions (unexpected)
		--- FAIL: TestNestedAttributesOnAHasManyAssociation#test_can_use_symbols_as_object_identifier (unexpected)
		--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyTest#test_references_foreign_key_with_suffix (unexpected)
		--- FAIL: BasicsTest#test_distinct_delegates_to_scoped (unexpected)
		--- FAIL: PersistenceTest#test_update_attribute_does_not_run_sql_if_attribute_is_not_changed (unexpected)
		--- FAIL: PersistenceTest#test_save_invalid_record (unexpected)
		--- FAIL: PersistenceTest#test_update (unexpected)
		--- FAIL: PersistenceTest#test_update_attribute (unexpected)
		--- FAIL: TestAutosaveAssociationOnAHasManyAssociation#test_should_not_update_children_when_parent_creation_with_no_reason (unexpected)
		--- FAIL: PersistenceTest#test_becomes_wont_break_mutation_tracking (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_create_polymorphic_has_one_with_scope (unexpected)
		--- FAIL: ReservedWordTest#test_associations_work_with_reserved_words (unexpected)
		--- FAIL: CockroachDB::FixturesResetPkSequenceTest#test_create_fixtures_resets_sequences_when_not_cached (unexpected)
		--- FAIL: PersistenceTest#test_update_does_not_run_sql_if_record_has_not_changed (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_create_associate_when_adding_to_has_many_through (unexpected)
		--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_object_arg (unexpected)
		--- FAIL: PersistenceTest#test_persisted_returns_boolean (unexpected)
		--- FAIL: PersistenceTest#test_update_attribute_for_updated_at_on (unexpected)
		--- FAIL: BasicsTest#test_#connecting_to_doesn't_work_with_legacy_connection_handling (unexpected)
		--- FAIL: StrictLoadingTest#test_strict_loading_by_default_can_be_set_per_model (unexpected)
		--- FAIL: BasicsTest#test_previously_new_record_returns_boolean (unexpected)
		--- FAIL: PersistenceTest#test_update_all (unexpected)
		--- FAIL: BasicsTest#test_create_after_initialize_with_block (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_inferes_column (unexpected)
		--- FAIL: PersistenceTest#test_update_parameters (unexpected)
		--- FAIL: BasicsTest#test_clone_of_new_object_marks_attributes_as_dirty (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: PersistenceTest#test_raises_error_when_validations_failed (unexpected)
		--- FAIL: StrictLoadingTest#test_strict_loading_with_has_many (unexpected)
		--- FAIL: BasicsTest#test_#connected_to_many_sets_prevent_writes_if_role_is_reading (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- FAIL: EnumTest#test_validate_uniqueness (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_column (unexpected)
		--- FAIL: PersistenceTest#test_update_raises_record_not_found_exception (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_distinct_through_count (unexpected)
		--- FAIL: StrictLoadingTest#test_strict_loading_with_reflection_is_ignored_in_validation_context (unexpected)
		--- FAIL: InheritanceComputeTypeTest#test_sti_type_from_attributes_disabled_in_non_sti_class (unexpected)
		--- FAIL: AssociationsTest#test_association_with_references (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5915393?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5915393?buildTab=artifacts#/activerecord) on master @ [cd113bf428e8868c38b2f2f295210f5c30d13a2d](https://github.com/cockroachdb/cockroach/commits/cd113bf428e8868c38b2f2f295210f5c30d13a2d):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2040-gcd113bf428
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5921532?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5921532?buildTab=artifacts#/activerecord) on master @ [0352fbee5e6933e8bca28aae4b136bcd495ab227](https://github.com/cockroachdb/cockroach/commits/0352fbee5e6933e8bca28aae4b136bcd495ab227):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2042-g0352fbee5e
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5926514?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5926514?buildTab=artifacts#/activerecord) on master @ [0352fbee5e6933e8bca28aae4b136bcd495ab227](https://github.com/cockroachdb/cockroach/commits/0352fbee5e6933e8bca28aae4b136bcd495ab227):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2042-g0352fbee5e
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7352 tests passed
		20 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		20 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_change_column_of_parent_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_change_column_of_parent_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5928734?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5928734?buildTab=artifacts#/activerecord) on master @ [b83e83b7e54fabf6ebd3a42cff67f4b8c5b7b863](https://github.com/cockroachdb/cockroach/commits/b83e83b7e54fabf6ebd3a42cff67f4b8c5b7b863):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2049-gb83e83b7e5
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5954907?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5954907?buildTab=artifacts#/activerecord) on master @ [7326bbbf94610c40ae2d2fdf42c119dfea93a170](https://github.com/cockroachdb/cockroach/commits/7326bbbf94610c40ae2d2fdf42c119dfea93a170):


```
		--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association_with_conditions_on_nested_association (unexpected)
		--- FAIL: FinderTest#test_model_class_responds_to_fifth_bang (unexpected)
		--- FAIL: BasicsTest#test_marshalling_with_associations (unexpected)
		--- FAIL: EagerAssociationTest#test_preconfigured_includes_with_belongs_to (unexpected)
		--- FAIL: EagerAssociationTest#test_including_association_based_on_sql_condition_and_no_database_column (unexpected)
		--- FAIL: EagerAssociationTest#test_loading_association_with_same_table_joins (unexpected)
		--- FAIL: SerializedAttributeTest#test_serialized_no_default_class_for_object (unexpected)
		--- FAIL: RelationTest#test_relations_show_the_records_in_#inspect (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_has_and_belongs_to_many_in_a_namespaced_model_pointing_to_a_namespaced_model (unexpected)
		--- FAIL: EagerAssociationTest#test_loading_polymorphic_association_with_mixed_table_conditions (unexpected)
		--- FAIL: RequiredAssociationsTest#test_required_has_one_associations_have_a_correct_error_message (unexpected)
		--- FAIL: FinderTest#test_find_by_with_range_conditions_returns_the_first_matching_record (unexpected)
		--- FAIL: FinderTest#test_exists (unexpected)
		--- FAIL: FinderTest#test_custom_select_takes_precedence_over_original_value (unexpected)
		--- FAIL: RelationTest#test_one_with_destroy (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_should_set_foreign_key_on_create_association (unexpected)
		--- FAIL: FinderTest#test_find_by_sql_with_sti_on_joined_table (unexpected)
		--- FAIL: FinderTest#test_bind_variables_with_quotes (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_and_limit (unexpected)
		--- FAIL: FinderTest#test_find_on_relation_with_large_number (unexpected)
		--- FAIL: RelationTest#test_primary_key (unexpected)
		--- FAIL: BasicsTest#test_slice_accepts_array_argument (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_where_on_polymorphic_association_with_empty_array (unexpected)
		--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association_with_order_on_nested_association (unexpected)
		--- FAIL: FinderTest#test_condition_interpolation (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_adding_from_the_project_fixed_timestamp (unexpected)
		--- FAIL: EagerAssociationTest#test_limited_eager_with_numeric_in_association (unexpected)
		--- FAIL: SerializedAttributeTestWithYamlSafeLoad#test_json_read_legacy_null (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_invalid_association_reference (unexpected)
		--- FAIL: InheritanceTest#test_compute_type_on_undefined_method (unexpected)
		--- FAIL: EagerAssociationTest#test_preloading_a_regular_association_with_a_typo_through_a_polymorphic_association_still_raises (unexpected)
		--- FAIL: InheritanceTest#test_different_namespace_subclass_should_load_correctly_with_store_full_sti_class_option (unexpected)
		--- FAIL: FinderTest#test_find_by_returns_nil_if_the_record_is_missing (unexpected)
		--- FAIL: RelationTest#test_loaded_first (unexpected)
		--- FAIL: RelationTest#test_dynamic_finder (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_with_custom_foreign_type (unexpected)
		--- FAIL: FinderTest#test_hash_condition_find_malformed (unexpected)
		--- FAIL: BasicsTest#test_previously_new_record_returns_boolean (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_destroying_polymorphic_child_with_unloaded_parent_and_touch_is_possible_with_has_many_inversing (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
		--- FAIL: EagerAssociationTest#test_loading_with_scope_including_joins (unexpected)
		--- FAIL: EagerAssociationTest#test_eager-loading_non-readonly_association (unexpected)
		--- FAIL: EagerAssociationTest#test_finding_with_includes_on_has_one_association_with_same_include_includes_only_once (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_eager_loading_with_primary_key_as_symbol (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_find_should_append_to_association_order (unexpected)
		--- FAIL: SerializedAttributeTestWithYamlSafeLoad#test_values_cast_from_nil_are_persisted_as_nil (unexpected)
		--- FAIL: InheritanceTest#test_should_store_full_class_name_with_store_full_sti_class_option_enabled (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5969340?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5969340?buildTab=artifacts#/activerecord) on master @ [4d40b770f96e8248a937b614c5909289c38c777a](https://github.com/cockroachdb/cockroach/commits/4d40b770f96e8248a937b614c5909289c38c777a):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2179-g4d40b770f9
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5981014?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5981014?buildTab=artifacts#/activerecord) on master @ [d687b14e513a49f94e1862c78a7685b78f4fbad9](https://github.com/cockroachdb/cockroach/commits/d687b14e513a49f94e1862c78a7685b78f4fbad9):


```
		--- FAIL: FinderTest#test_find_with_order_on_included_associations_with_construct_finder_sql_for_association_limiting_and_is_distinct (unexpected)
		--- FAIL: FinderTest#test_find_by_with_multi-arg_conditions_returns_the_first_matching_record (unexpected)
		--- FAIL: FinderTest#test_model_class_responds_to_third_to_last_bang (unexpected)
		--- FAIL: FinderTest#test_include_on_unloaded_relation_without_match (unexpected)
		--- FAIL: FinderTest#test_third_with_offset (unexpected)
		--- FAIL: FinderTest#test_take_bang_missing (unexpected)
		--- FAIL: FinderTest#test_find_by_association_subquery (unexpected)
		--- FAIL: PessimisticLockingTest#test_locking_in_after_save_callback (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: FinderTest#test_unexisting_record_exception_handling (unexpected)
		--- FAIL: PostgresqlInfinityTest#test_where_clause_with_infinite_range_on_a_datetime_column (unexpected)
		--- FAIL: EagerAssociationTest#test_base_messages (unexpected)
		--- FAIL: FinderTest#test_symbols_table_ref (unexpected)
		--- FAIL: EagerAssociationTest#test_has_many_association_ignores_the_scoping (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_store_association_in_two_relations_with_one_save_in_existing_object_with_values (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_valid_association_as_string_not_symbol (unexpected)
		--- FAIL: FinderTest#test_second_have_primary_key_order_by_default (unexpected)
		--- FAIL: FinderTest#test_member_on_unloaded_relation_with_mismatched_class (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_loading_with_order_on_joined_table_preloads (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_default_scope_as_callable (unexpected)
		--- FAIL: FinderTest#test_include_on_loaded_relation_with_match (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_foreign_keys (unexpected)
		--- FAIL: FinderTest#test_fourth_have_primary_key_order_by_default (unexpected)
		--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_2#test_create_join_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
		--- FAIL: FinderTest#test_find_by_one_attribute_bang_with_blank_defined (unexpected)
		--- FAIL: EagerAssociationTest#test_eager-loading_with_a_polymorphic_association_won't_work_consistently (unexpected)
		--- FAIL: FinderTest#test_find_with_proc_parameter_and_block (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: FinderTest#test_find_by_on_attribute_that_is_a_reserved_word (unexpected)
		--- FAIL: FinderTest#test_find_with_nil_inside_set_passed_for_attribute (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_going_through_join_model_with_inheritance (unexpected)
		--- FAIL: FinderTest#test_model_class_responds_to_first_bang (unexpected)
		--- FAIL: EagerAssociationTest#test_has_many_through_with_order (unexpected)
		--- FAIL: FinderTest#test_bind_variables (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_unabstracted_database_dependent_types (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_include_has_many_through (unexpected)
		--- FAIL: FinderTest#test_find_by_id_with_conditions_with_or (unexpected)
		--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association_with_order (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_going_through_join_model_with_inheritance_with_custom_class_name (unexpected)
		--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_0#test_change_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
		--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association_with_order_on_nested_association (unexpected)
		--- FAIL: FinderTest#test_find_on_hash_conditions_with_array_of_integers_and_ranges (unexpected)
		--- FAIL: FinderTest#test_find_an_empty_array (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_load_belongs_to_with_string_keys (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: FinderTest#test_find_on_hash_conditions_with_range (unexpected)
		--- FAIL: ActiveRecordSchemaTest#test_schema_load_with_multiple_indexes_for_column_of_different_names (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5991660?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5991660?buildTab=artifacts#/activerecord) on master @ [db95c2484a9f15c2de01466af0aae19c8ab57618](https://github.com/cockroachdb/cockroach/commits/db95c2484a9f15c2de01466af0aae19c8ab57618):


```
		--- FAIL: CascadedEagerLoadingTest#test_preloaded_records_are_not_duplicated (unexpected)
		--- FAIL: RelationTest#test_create_with_array (unexpected)
		--- FAIL: HasManyAssociationsTest#test_deleting_updates_counter_cache_with_dependent_destroy (unexpected)
		--- FAIL: FinderTest#test_find_passing_active_record_object_is_not_permitted (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_polymorphic_belongs_to (unexpected)
		--- FAIL: HasManyAssociationsTest#test_include_returns_false_for_non_matching_record_to_verify_scoping (unexpected)
		--- FAIL: HasManyAssociationsTest#test_get_ids (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_belongs_to_sti (unexpected)
		--- FAIL: RelationTest#test_grouping_by_column_with_reserved_name (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_deleting_by_integer_id_from_has_many_through (unexpected)
		--- FAIL: RelationTest#test_order_with_hash_and_symbol_generates_the_same_sql (unexpected)
		--- FAIL: RelationTest#test_#where_with_empty_set (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_grouped (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: RelationTest#test_annotate_with_blank_value (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_eager_load_has_many_through_has_many (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dependent_association_respects_optional_sanitized_conditions_on_delete (unexpected)
		--- FAIL: MultipleDbTest#test_course_connection_should_survive_reloads (unexpected)
		--- FAIL: RelationTest#test_create_or_find_by_should_not_raise_due_to_validation_errors (unexpected)
		--- FAIL: HasManyAssociationsTest#test_included_in_collection (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_ids_and_inverse_of (unexpected)
		--- FAIL: HasManyAssociationsTest#test_include_uses_array_include_after_loaded (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_going_through_join_model_with_custom_primary_key (unexpected)
		--- FAIL: FinderTest#test_exists_with_empty_hash_arg (unexpected)
		--- FAIL: HasManyAssociationsTest#test_get_ids_for_through (unexpected)
		--- FAIL: RelationTest#test_automatically_added_reorder_references (unexpected)
		--- FAIL: RelationTest#test_where_with_take_memoization (unexpected)
		--- FAIL: RelationTest#test_where_id_with_delegated_ar_object (unexpected)
		--- FAIL: RelationTest#test_pluck_with_from_includes_quoted_original_table_name (unexpected)
		--- FAIL: FinderTest#test_hash_condition_find_with_aggregate_having_three_mappings_array (unexpected)
		--- FAIL: RelationTest#test_except (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_delete_polymorphic_has_many_with_destroy (unexpected)
		--- FAIL: FinderTest#test_hash_condition_find_empty_array_with_aggregate_having_multiple_mappings (unexpected)
		--- FAIL: FinderTest#test_exists_with_string (unexpected)
		--- FAIL: RelationTest#test_finding_with_conditions (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_unavailable_through_reflection (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_first_after_reload (unexpected)
		--- FAIL: FinderTest#test_hash_condition_find_nil_with_aggregate_having_multiple_mappings (unexpected)
		--- FAIL: FinderTest#test_take_bang_present (unexpected)
		--- FAIL: RelationTest#test_doesnt_add_having_values_if_options_are_blank (unexpected)
		--- FAIL: FinderTest#test_joins_with_string_array (unexpected)
		--- FAIL: FinderTest#test_find_by_invalid_method_syntax (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_delete_associate_when_deleting_from_has_many_through_with_nonstandard_id (unexpected)
		--- FAIL: PreloaderTest#test_some_already_loaded_associations (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_first_or_last_with_integer_on_association_should_not_load_association (unexpected)
		--- FAIL: FinderTest#test_find_by_id_with_conditions_with_or (unexpected)
		--- FAIL: FinderTest#test_eager_load_for_no_has_many_with_limit_and_left_joins_for_has_many (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5997534?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5997534?buildTab=artifacts#/activerecord) on master @ [a7c91f06d8ee0fa2096bcd626f689009024947bb](https://github.com/cockroachdb/cockroach/commits/a7c91f06d8ee0fa2096bcd626f689009024947bb):


```
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_suffix (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_touch_without_updated_at_attributes (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_primary_key_counter (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_assignment_with_primary_key_foreign_type_field_updating (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_where_on_polymorphic_association_with_nil (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_create_with_block (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_create! (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_assignment_before_child_saved_with_primary_key (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: TouchLaterTest#test_touch_touches_immediately_with_a_custom_time (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_creating_the_belonging_object_with_primary_key (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_default_with_lambda (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_cant_save_readonly_association (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_counter_with_assigning_new_object (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_find_by_with_custom_primary_key (unexpected)
		--- FAIL: CockroachDB::FixturesTest#test_yaml_file_with_symbol_columns (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_assigning_belongs_to_on_destroyed_object (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_constraint_by_name (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_dependency_should_halt_parent_destruction_with_cascaded_three_levels (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
		--- FAIL: CockroachDB::MigrationTest#test_remove_column_with_if_not_exists_not_set (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_eager_loading_with_primary_key_as_symbol (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_where_with_custom_primary_key (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_nullify_foreign_key (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_name (unexpected)
		--- FAIL: TouchLaterTest#test_touching_three_deep (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_id_assigning (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_inferes_column (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_attributes_are_being_set_when_initialized_from_belongs_to_association_with_where_clause (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_without_counter_cache_option (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_save_of_record_with_loaded_belongs_to (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_not_optional_relation (unexpected)
		--- FAIL: TouchLaterTest#test_touch_later_raise_if_non_persisted (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_destroying_child_with_unloaded_parent_and_foreign_key_and_touch_is_possible_with_has_many_inversing (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_name (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_symbol_column (unexpected)
		--- FAIL: TouchLaterTest#test_touch_later_an_association_dont_autosave_parent (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_works_with_model_called_Record (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_assignment_with_nil (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_setting_foreign_key_after_nil_target_loaded (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_building_the_belonging_object_with_explicit_sti_base_class (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_proxy_assignment (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6004536?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6004536?buildTab=artifacts#/activerecord) on master @ [ce8bfd4811e789bdd4cf31640722517a5cf46237](https://github.com/cockroachdb/cockroach/commits/ce8bfd4811e789bdd4cf31640722517a5cf46237):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2273-gce8bfd4811
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7363 tests passed
		9 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		9 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: QueryCacheTest#test_query_cache_does_not_allow_sql_key_mutation (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6019684?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6019684?buildTab=artifacts#/activerecord) on master @ [9a0a233f4021f25c1766434bb6aee85536e32036](https://github.com/cockroachdb/cockroach/commits/9a0a233f4021f25c1766434bb6aee85536e32036):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2326-g9a0a233f40
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6034844?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6034844?buildTab=artifacts#/activerecord) on master @ [f53c21f9fe0fe07ba32d2f28ac4bd2c4bc2ed50b](https://github.com/cockroachdb/cockroach/commits/f53c21f9fe0fe07ba32d2f28ac4bd2c4bc2ed50b):


```
		--- FAIL: TestNestedAttributesOnAHasManyAssociation#test_should_take_an_array_and_assign_the_attributes_to_the_associated_models (unexpected)
		--- FAIL: ActiveRecord::Migration::CommandRecorderTest#test_invert_drop_join_table (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_multiple_associations_with_same_table_has_one (unexpected)
		--- FAIL: HasManyAssociationsTest#test_defining_has_many_association_with_nullify_dependency_lazily_evaluates_target_class (unexpected)
		--- FAIL: HasManyAssociationsTestPrimaryKeys#test_ids_on_unloaded_association_with_custom_primary_key (unexpected)
		--- FAIL: NestedThroughAssociationsTest#test_has_many_through_has_many_with_has_many_through_habtm_source_reflection_preload_via_joins (unexpected)
		--- FAIL: RelationTest#test_order_using_scoping (unexpected)
		--- FAIL: ActiveRecord::Migration::ReferencesStatementsTest#test_primary_key_and_references_columns_should_be_identical_type (unexpected)
		--- FAIL: PrimaryKeysTest#test_primary_key_returns_nil_if_it_does_not_exist (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_association_with_validate_false_does_not_run_associated_validation_callbacks_on_create (unexpected)
		--- FAIL: StoreTestWithYAMLSafeLoad#test_new_record_and_no_accessors_changes (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_name (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_create_model_with_inheritance (unexpected)
		--- FAIL: TestNestedAttributesOnAHasOneAssociation#test_should_work_with_update_as_well (unexpected)
		--- FAIL: StoreTestWithYAMLSafeLoad#test_all_stored_attributes_are_returned (unexpected)
		--- FAIL: ActiveRecord::Relation::WhereClauseTest#test_or_places_common_conditions_before_the_OR (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_building_the_belonging_object (unexpected)
		--- FAIL: SchemaDumperTest#test_schema_dump_with_correct_timestamp_types_via_create_table_and_t_timestamptz (unexpected)
		--- FAIL: ActiveRecord::OrTest#test_or_with_null_right (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_ids_and_inverse_of (unexpected)
		--- FAIL: PreloaderTest#test_preload_with_available_records_queries_when_scoped (unexpected)
		--- FAIL: InnerJoinAssociationTest#test_join_with_reserved_word (unexpected)
		--- FAIL: EachTest#test_each_should_return_an_enumerator_if_no_block_is_present (unexpected)
		--- FAIL: HasManyAssociationsTest#test_depends_and_nullify_on_polymorphic_assoc (unexpected)
		--- FAIL: ActiveRecord::Migration::CommandRecorderTest#test_invert_enable_extension (unexpected)
		--- FAIL: SignedIdTest#test_find_signed_record_with_a_bang_on_relation (unexpected)
		--- FAIL: InnerJoinAssociationTest#test_user_supplied_joins_order_should_be_preserved (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_conditions_string_with_unquoted_table_name (unexpected)
		--- FAIL: CalculationsTest#test_should_return_nil_as_average (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_calculations_on_has_many_through_should_disambiguate_fields (unexpected)
		--- FAIL: CalculationsTest#test_distinct_count_all_with_custom_select_and_order (unexpected)
		--- FAIL: ActiveRecord::Encryption::ConfigurableTest#test_exclude_the_installation_of_autofiltered_params (unexpected)
		--- FAIL: InsertAllTest#test_insert_all_and_upsert_all_with_expression_index (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_reassignment_of_associated_id_updates_the_object (unexpected)
		--- FAIL: NestedThroughAssociationsTest#test_nested_has_many_through_should_not_be_autosaved (unexpected)
		--- FAIL: CalculationsTest#test_pluck_auto_table_name_prefix (unexpected)
		--- FAIL: HasOneThroughAssociationsTest#test_creating_association_creates_through_record (unexpected)
		--- FAIL: EachTest#test_in_batches_each_record_should_return_enumerator_if_no_block_given (unexpected)
		--- FAIL: HasManyAssociationsTest#test_build_with_polymorphic_has_many_does_not_allow_to_override_type_and_id (unexpected)
		--- FAIL: CockroachDB::PostgresqlNumberTest#test_decimal_values (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: RelationTest#test_#where_with_set (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_default_scope_as_lambda (unexpected)
		--- FAIL: SerializedAttributeTestWithYamlSafeLoad#test_nil_is_not_changed_when_serialized_with_a_class (unexpected)
		--- FAIL: HasManyAssociationsTest#test_select_query_method (unexpected)
		--- FAIL: ActiveRecord::ConnectionAdapters::ConnectionHandlersMultiDbTest#test_loading_relations_with_multi_db_connections (unexpected)
		--- FAIL: ActiveRecord::Migration::CommandRecorderTest#test_invert_remove_column (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6048219?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6048219?buildTab=artifacts#/activerecord) on master @ [43bfc3185cb362b14e0b8a331148f0b14542a4f2](https://github.com/cockroachdb/cockroach/commits/43bfc3185cb362b14e0b8a331148f0b14542a4f2):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2434-g43bfc3185c
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6064717?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6064717?buildTab=artifacts#/activerecord) on master @ [773f7d4445ce3e0e806b7a182adba70a0f270f19](https://github.com/cockroachdb/cockroach/commits/773f7d4445ce3e0e806b7a182adba70a0f270f19):


```
		--- FAIL: PersistenceTest#test_persisted_returns_boolean (unexpected)
		--- FAIL: PersistenceTest#test_decrement_with_touch_updates_timestamps (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: TouchLaterTest#test_touch_later_raise_if_non_persisted (unexpected)
		--- FAIL: CounterCacheTest#test_counter_caches_are_updated_in_memory_when_the_default_value_is_nil (unexpected)
		--- FAIL: PersistenceTest#test_update! (unexpected)
		--- FAIL: PersistenceTest#test_class_level_update_is_affected_by_scoping! (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: BasicsTest#test_incomplete_schema_loading (unexpected)
		--- FAIL: PersistenceTest#test_update_many! (unexpected)
		--- FAIL: HasOneAssociationsTest#test_reload_association (unexpected)
		--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_rollback_destructions_if_an_exception_occurred_while_saving_a_parent (unexpected)
		--- FAIL: BasicsTest#test_marshal_new_record_round_trip (unexpected)
		--- FAIL: CounterCacheTest#test_reset_counters_with_modularized_and_camelized_classnames (unexpected)
		--- FAIL: HasOneAssociationsTest#test_natural_assignment_to_nil_after_destroy (unexpected)
		--- FAIL: HasOneAssociationsTest#test_polymorphic_has_one_with_touch_option_on_create_wont_cache_association_so_fetching_after_transaction_commit_works (unexpected)
		--- FAIL: TouchLaterTest#test_touch_later_dont_set_dirty_attributes (unexpected)
		--- FAIL: HasOneAssociationsTest#test_create_respects_hash_condition (unexpected)
		--- FAIL: BasicsTest#test_bignum_pk (unexpected)
		--- FAIL: PersistenceTest#test_update (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: HasOneAssociationsTest#test_exclusive_dependence (unexpected)
		--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_save_changed_child_objects_if_parent_is_saved (unexpected)
		--- FAIL: PersistenceTest#test_update_many_with_active_record_base_object (unexpected)
		--- FAIL: BasicsTest#test_many_mutations (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_set (unexpected)
		--- FAIL: BasicsTest#test_resetting_column_information_doesn't_remove_attribute_methods (unexpected)
		--- FAIL: PersistenceTest#test_save_with_duping_of_destroyed_object (unexpected)
		--- FAIL: BasicsTest#test_preserving_time_objects_with_time_with_zone_conversion_to_default_timezone_utc (unexpected)
		--- FAIL: CockroachDB::WithAnnotationsTest#test_has_and_belongs_to_many_with_annotation_includes_a_query_comment (unexpected)
		--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_still_raise_an_ActiveRecordRecord_Invalid_exception_if_we_want_that (unexpected)
		--- FAIL: HasOneAssociationsTest#test_has_one_with_touch_option_on_update (unexpected)
		--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_save_new_record_that_has_same_value_as_existing_record_marked_for_destruction_on_field_that_has_unique_index (unexpected)
		--- FAIL: CounterCacheTest#test_update_counters_in_a_polymorphic_relationship (unexpected)
		--- FAIL: HasManyAssociationsTest#test_has_many_without_counter_cache_option (unexpected)
		--- FAIL: BasicsTest#test_slice_accepts_array_argument (unexpected)
		--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_destroy_has_many_as_part_of_the_save_transaction_if_they_were_marked_for_destruction (unexpected)
		--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_save_changed_has_one_changed_object_if_child_is_saved (unexpected)
		--- FAIL: BasicsTest#test_find_by_slug_with_array (unexpected)
		--- FAIL: CounterCacheTest#test_decrement_counter (unexpected)
		--- FAIL: PersistenceTest#test_destroy_raises_record_not_found_exception (unexpected)
		--- FAIL: BasicsTest#test_attributes (unexpected)
		--- FAIL: BasicsTest#test_equality_of_relation_and_collection_proxy (unexpected)
		--- FAIL: BasicsTest#test_benchmark_with_use_silence (unexpected)
		--- FAIL: BasicsTest#test_set_table_name_symbol_converted_to_string (unexpected)
		--- FAIL: BasicsTest#test_#connecting_to_with_role_and_shard (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6087204?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6087204?buildTab=artifacts#/activerecord) on master @ [d25cb57ccd9bc643ce9058ebd2057cab36b69ad5](https://github.com/cockroachdb/cockroach/commits/d25cb57ccd9bc643ce9058ebd2057cab36b69ad5):


```
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_properties_assigned_in_constructor (unexpected)
		--- FAIL: StrictLoadingTest#test_raises_on_lazy_loading_a_strict_loading_belongs_to_relation (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_invalid_foreign_key (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_immutable_strings_by_default_does_not_affect_``attribute_:foo,_:string`` (unexpected)
		--- FAIL: StrictLoadingTest#test_strict_loading_is_ignored_in_validation_context (unexpected)
		--- FAIL: StrictLoadingTest#test_strict_loading_can_be_turned_off_on_an_association_in_a_model_with_strict_loading_on (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_validation_does_not_validate_non_dirty_association_target (unexpected)
		--- FAIL: StrictLoadingTest#test_preload_audit_logs_are_strict_loading_because_it_is_strict_loading_by_default (unexpected)
		--- FAIL: ActiveRecord::AdapterForeignKeyTest#test_disable_referential_integrity (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_on_update (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_overloading_types (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_caches_are_cleared (unexpected)
		--- FAIL: UpdateAllTest#test_update_all_with_joins (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_range_types_can_be_specified (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_add_column_with_precision_and_scale (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists (unexpected)
		--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_limit_and_utf8 (unexpected)
		--- FAIL: UpdateAllTest#test_update_all_with_blank_argument (unexpected)
		--- FAIL: PostgresqlInfinityTest#test_type_casting_infinity_on_a_date_column (unexpected)
		--- FAIL: StrictLoadingTest#test_does_not_raise_on_eager_loading_a_strict_loading_habtm_relation (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_native_types (unexpected)
		--- FAIL: HabtmDestroyOrderTest#test_should_not_raise_error_if_have_foreign_key_in_the_join_table (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_attributes_do_not_require_a_type (unexpected)
		--- FAIL: UpdateAllTest#test_update_all_ignores_order_without_limit_from_association (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_set (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_attributes_not_backed_by_database_columns_are_always_initialized (unexpected)
		--- FAIL: AssociationsExtensionsTest#test_proxy_association_after_scoped (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_cascade_foreign_key (unexpected)
		--- FAIL: AssociationsExtensionsTest#test_extension_on_has_many (unexpected)
		--- FAIL: UpdateAllTest#test_update_all_with_group_by (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_column (unexpected)
		--- FAIL: UpdateAllTest#test_update_counters_with_joins (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_native_decimal_insert_manual_vs_automatic (unexpected)
		--- FAIL: StrictLoadingTest#test_strict_loading_violation_can_log_instead_of_raise (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_symbol_column (unexpected)
		--- FAIL: UpdateAllTest#test_update_on_relation_passing_active_record_object_is_not_permitted (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: StrictLoadingTest#test_strict_loading_with_has_many_through_cascade_down_to_middle_records (unexpected)
		--- FAIL: ActiveRecord::Migration::CheckConstraintTest#test_not_valid_check_constraint (unexpected)
		--- FAIL: CockroachDB::EagerLoadPolyAssocsTest#test_include_query (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_singular_ids_are_reloaded_after_collection_concat (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_inferes_column (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: AssociationsExtensionsTest#test_extension_name (unexpected)
		--- FAIL: AssociationsExtensionsTest#test_named_extension_and_block_on_habtm (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_symbol_column (unexpected)
		--- FAIL: PostgresqlInfinityTest#test_type_casting_string_on_a_float_column (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6094282?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6094282?buildTab=artifacts#/activerecord) on master @ [41db784cb97d2749b162020c2c821979094f87b1](https://github.com/cockroachdb/cockroach/commits/41db784cb97d2749b162020c2c821979094f87b1):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2646-g41db784cb9
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7357 tests passed
		15 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		15 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_change_column_of_parent_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: BasicsTest#test_default_values_are_deeply_dupped (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6107800?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6107800?buildTab=artifacts#/activerecord) on master @ [f4042d47fa8062a612c38d4696eb6bee9cee7c21](https://github.com/cockroachdb/cockroach/commits/f4042d47fa8062a612c38d4696eb6bee9cee7c21):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2689-gf4042d47fa
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6124150?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6124150?buildTab=artifacts#/activerecord) on master @ [b173a16715e71e94115820374da1eb350b3b459d](https://github.com/cockroachdb/cockroach/commits/b173a16715e71e94115820374da1eb350b3b459d):


```
		--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_merge_errors_on_the_associated_models_onto_the_parent_even_if_it_is_not_valid (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_natural_assignment (unexpected)
		--- FAIL: HasManyAssociationsTest#test_in_memory_replacement_maintains_order (unexpected)
		--- FAIL: CockroachDB::WithAnnotationsTest#test_has_many_through_with_annotation_includes_a_query_comment_when_eager_loading (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dependence_with_transaction_support_on_failure (unexpected)
		--- FAIL: EagerAssociationTest#test_preloading_has_many_through_association_avoids_calling_association.reader (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to (unexpected)
		--- FAIL: RelationScopingTest#test_reverse_order_with_arel_node (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dependent_association_respects_optional_sanitized_conditions_on_delete (unexpected)
		--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_not_use_default_invalid_error_on_associated_models (unexpected)
		--- FAIL: EagerAssociationTest#test_loading_with_one_association_with_non_preload (unexpected)
		--- FAIL: ReservedWordTest#test_belongs_to_associations (unexpected)
		--- FAIL: EagerAssociationTest#test_loading_with_scope_including_joins (unexpected)
		--- FAIL: HasManyAssociationsTest#test_has_many_without_counter_cache_option (unexpected)
		--- FAIL: EagerAssociationTest#test_should_work_inverse_of_with_eager_load (unexpected)
		--- FAIL: HasManyAssociationsTest#test_transaction_when_deleting_persisted (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_through_an_sti_join_model (unexpected)
		--- FAIL: ReservedWordTest#test_calculations_work_with_reserved_words (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_set (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_counter_with_assigning_nil (unexpected)
		--- FAIL: HasManyAssociationsTest#test_destroy_all (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_column (unexpected)
		--- FAIL: HasManyAssociationsTest#test_deleting_updates_counter_cache_with_dependent_delete_all (unexpected)
		--- FAIL: FixturesTest#test_serialized_fixtures (unexpected)
		--- FAIL: EagerAssociationTest#test_attribute_alias_in_where_references_association_name (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_load_has_many_with_string_keys (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_through (unexpected)
		--- FAIL: EagerAssociationTest#test_eager-loading_with_a_polymorphic_association_won't_work_consistently (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_setting_foreign_key_after_nil_target_loaded (unexpected)
		--- FAIL: EagerAssociationTest#test_preloading_has_many_through_with_distinct (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_order_string_with_unquoted_table_name (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_assignment_foreign_key_type_string (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_load_has_many_through_quotes_table_and_column_names (unexpected)
		--- FAIL: HasManyAssociationsTest#test_replace_with_less_and_dependent_nullify (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_tracking_polymorphic_changes (unexpected)
		--- FAIL: EagerAssociationTest#test_association_loading_notification (unexpected)
		--- FAIL: HasManyAssociationsTest#test_get_ids_for_loaded_associations (unexpected)
		--- FAIL: StoreTestWithYAMLSafeLoad#test_updating_the_store_will_mark_it_as_changed (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_through_join_model_with_conditions (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_eager_loading_wont_mutate_owner_record (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_update (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_and_limit_and_conditions_array_on_the_eagers (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_loading_with_order_on_joined_table_preloads (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_native_decimal_insert_manual_vs_automatic (unexpected)
		--- FAIL: HasManyAssociationsTest#test_attributes_are_set_when_initialized_from_polymorphic_has_many_null_relationship (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_touch_without_updated_at_attributes (unexpected)
		--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_rollback_any_changes_if_an_exception_occurred_while_saving (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6137147?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6137147?buildTab=artifacts#/activerecord) on master @ [5c2c62ecf1bea60c807edc6b4da22d900ad4ae03](https://github.com/cockroachdb/cockroach/commits/5c2c62ecf1bea60c807edc6b4da22d900ad4ae03):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2800-g5c2c62ecf1
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7363 tests passed
		9 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		9 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_simple_interval (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6151881?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6151881?buildTab=artifacts#/activerecord) on master @ [a0d8839aa6164af81a9ebb140147d3baf5321287](https://github.com/cockroachdb/cockroach/commits/a0d8839aa6164af81a9ebb140147d3baf5321287):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2834-ga0d8839aa6
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6166285?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6166285?buildTab=artifacts#/activerecord) on master @ [cb55144cdec54d2a70f074ad64b4eca5e6c6891a](https://github.com/cockroachdb/cockroach/commits/cb55144cdec54d2a70f074ad64b4eca5e6c6891a):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2899-gcb55144cde
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7358 tests passed
		14 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		14 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_change_column_of_parent_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6174190?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6174190?buildTab=artifacts#/activerecord) on master @ [e6a7dc2f8ee39549e186bd05626c4c375b76fd04](https://github.com/cockroachdb/cockroach/commits/e6a7dc2f8ee39549e186bd05626c4c375b76fd04):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2938-ge6a7dc2f8e
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7358 tests passed
		14 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		14 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_change_column_of_parent_table (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6181118?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6181118?buildTab=artifacts#/activerecord) on master @ [aaf50e920ceff3c2863ab96b9e3614b8434b70a8](https://github.com/cockroachdb/cockroach/commits/aaf50e920ceff3c2863ab96b9e3614b8434b70a8):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-2943-gaaf50e920c
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7363 tests passed
		9 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		9 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::CheckConstraintTest#test_not_valid_check_constraint (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6195961?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6195961?buildTab=artifacts#/activerecord) on master @ [80c274877a917580af62be6eb0cd48c8c7ae9c08](https://github.com/cockroachdb/cockroach/commits/80c274877a917580af62be6eb0cd48c8c7ae9c08):


```
		--- FAIL: HasManyAssociationsTest#test_ids_reader_cache_not_used_for_size_when_association_is_dirty (unexpected)
		--- FAIL: InheritanceTest#test_should_store_full_class_name_with_store_full_sti_class_option_enabled (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_changing_defaults (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_building_the_belonging_object (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_missing_first_record (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_going_through_join_model_with_inheritance_with_custom_class_name (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_belongs_to_sti (unexpected)
		--- FAIL: FixturesTest#test_broken_yaml_exception (unexpected)
		--- FAIL: StrictLoadingTest#test_raises_on_lazy_loading_a_strict_loading_belongs_to_relation (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_option_on_update (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_out_of_range_value_assigning (unexpected)
		--- FAIL: HasManyAssociationsTest#test_destroy_all_on_association_clears_scope (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_one_should_count_instead_of_loading_association (unexpected)
		--- FAIL: HasManyAssociationsTest#test_collection_size_after_building (unexpected)
		--- FAIL: ReflectionTest#test_foreign_key (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_deleting (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_redefine_habtm (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_has_and_belongs_to_many_in_a_namespaced_model_pointing_to_a_namespaced_model (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_associations_with_conditions (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_create_model_with_inheritance_and_custom_base_class (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_create_with_conditions (unexpected)
		--- FAIL: HasManyAssociationsTest#test_create_from_association_with_nil_values_should_work (unexpected)
		--- FAIL: FixturesTest#test_no_args_returns_all (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_load_multiple_associations_with_references (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_polymorphic_nested_array_where (unexpected)
		--- FAIL: DefaultScopingTest#test_default_scope_runs_on_select (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_update_changing_ids_changes_the_counter_cache (unexpected)
		--- FAIL: AssociationsExtensionsTest#test_named_extension_on_habtm (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_attributes_do_not_require_a_type (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_default_scope_on_relations_is_not_cached (unexpected)
		--- FAIL: FixturesTest#test_yml_file_in_subdirectory (unexpected)
		--- FAIL: ReflectionTest#test_includes_accepts_strings (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: FixturesTest#test_complete_instantiation (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_counter (unexpected)
		--- FAIL: PostgresqlNumberTest#test_update (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_belongs_to_array_value_where (unexpected)
		--- FAIL: HasManyAssociationsTest#test_counter_cache_updates_in_memory_after_update_with_inverse_of_disabled (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_marshal_dump (unexpected)
		--- FAIL: HasManyAssociationsTest#test_build_and_create_from_association_should_respect_unscope_over_default_scope (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_replace_counter_cache (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_attributes_not_backed_by_database_columns_keep_their_type_when_a_default_value_is_configured_separately (unexpected)
		--- FAIL: HasManyAssociationsTest#test_create_resets_cached_counters (unexpected)
		--- FAIL: InheritanceTest#test_alt_find_first_within_inheritance (unexpected)
		--- FAIL: HasManyAssociationsTest#test_modifying_a_through_a_has_many_should_raise (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_find_all (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6213710?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6213710?buildTab=artifacts#/activerecord) on master @ [003c0360de8b64319b5f0f127b99be91dbdca8a3](https://github.com/cockroachdb/cockroach/commits/003c0360de8b64319b5f0f127b99be91dbdca8a3):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-3029-g003c0360de
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6231545?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6231545?buildTab=artifacts#/activerecord) on master @ [524fd14da3fefcd849f44a835cc5f88f5dbdadcc](https://github.com/cockroachdb/cockroach/commits/524fd14da3fefcd849f44a835cc5f88f5dbdadcc):


```
		--- FAIL: PersistenceTest#test_update_columns (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_through_has_many_find_conditions (unexpected)
		--- FAIL: ActiveRecord::Migration::CompatibilityTest#test_migration_does_remove_unnamed_index (unexpected)
		--- FAIL: FinderTest#test_exists_with_aggregate_having_three_mappings (unexpected)
		--- FAIL: FinderTest#test_exists_with_scope (unexpected)
		--- FAIL: PersistenceTest#test_becomes_default_sti_subclass (unexpected)
		--- FAIL: HasManyAssociationsTest#test_unscopes_the_default_scope_of_associated_model_when_used_with_include (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_size_on_an_association_that_has_not_been_loaded_performs_a_query (unexpected)
		--- FAIL: BasicsTest#test_find_by_slug (unexpected)
		--- FAIL: LogSubscriberTest#test_initializes_runtime (unexpected)
		--- FAIL: FinderTest#test_hash_condition_find_with_aggregate_having_one_mapping (unexpected)
		--- FAIL: BasicsTest#test_clone_of_new_object_marks_attributes_as_dirty (unexpected)
		--- FAIL: ActiveRecord::Encryption::KeyProviderTest#test_serves_a_single_key_for_encrypting_and_decrypting (unexpected)
		--- FAIL: FoxyFixturesTest#test_generates_unique_ids (unexpected)
		--- FAIL: TestNestedAttributesInGeneral#test_should_not_build_a_new_record_if_reject_all_blank_returns_false (unexpected)
		--- FAIL: NestedThroughAssociationsTest#test_nested_has_many_through_with_conditions_on_through_associations_preload (unexpected)
		--- FAIL: ActiveRecord::DatabaseSelectorTest#test_write_to_primary_with_exception (unexpected)
		--- FAIL: LogSubscriberTest#test_cached_queries (unexpected)
		--- FAIL: ActiveRecord::ConnectionAdapters::LegacyConnectionHandlersShardingDbTest#test_swapping_shards_and_roles_in_a_multi_threaded_environment (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dependence (unexpected)
		--- FAIL: ReservedWordTest#test_delete_all_with_subselect (unexpected)
		--- FAIL: AssociationValidationTest#test_validates_presence_of_belongs_to_association__parent_is_new_record (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_store_association_in_two_relations_with_one_save (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_one (unexpected)
		--- FAIL: CallbacksTest#test_destroy (unexpected)
		--- FAIL: ActiveRecord::Migration::TableTest#test_references_column_type_with_polymorphic_and_options_null_is_false_adds_table_flag (unexpected)
		--- FAIL: HasManyAssociationsTest#test_include_uses_array_include_after_loaded (unexpected)
		--- FAIL: FinderTest#test_find_by_array_of_one_id (unexpected)
		--- FAIL: FinderTest#test_find_on_association_proxy_conditions (unexpected)
		--- FAIL: LintTest#test_errors_aref (unexpected)
		--- FAIL: FinderTest#test_include_on_unloaded_relation_with_having_referencing_aliased_select (unexpected)
		--- FAIL: ActiveRecord::DatabaseSelectorTest#test_can_write_while_reading_from_replicas_if_explicit (unexpected)
		--- FAIL: UpdateAllTest#test_touch_all_updates_records_timestamps (unexpected)
		--- FAIL: EagerSingularizationTest#test_eager_no_extra_singularization_has_and_belongs_to_many (unexpected)
		--- FAIL: HasManyAssociationsTest#test_build_many_via_block (unexpected)
		--- FAIL: StoreTestWithYAMLSafeLoad#test_object_initialization_with_not_nullable_column_encoded_with_JSON (unexpected)
		--- FAIL: ActiveRecord::ConnectionAdapters::ConnectionPoolTest#test_active_connection? (unexpected)
		--- FAIL: BasicsTest#test_cannot_call_connected_to_with_role_and_shard_on_non-abstract_classes (unexpected)
		--- FAIL: BasicsTest#test_column_names_are_quoted_when_using_#from_clause_and_model_has_ignored_columns (unexpected)
		--- FAIL: ActiveRecord::DatabaseTaskCheckTargetVersionTest#test_check_target_version_does_not_raise_error_on_empty_version (unexpected)
		--- FAIL: NestedThroughAssociationsTest#test_has_many_through_reset_source_reflection_after_loading_is_complete (unexpected)
		--- FAIL: BasicsTest#test_bignum (unexpected)
		--- FAIL: DirtyTest#test_partial_insert_with_empty_values (unexpected)
		--- FAIL: FinderTest#test_hash_condition_utc_time_interpolation_with_default_timezone_local (unexpected)
		--- FAIL: BasicsTest#test_custom_mutator (unexpected)
		--- FAIL: FinderTest#test_#skip_query_cache!_for_#exists? (unexpected)
		--- FAIL: SerializedAttributeTest#test_serialized_boolean_value_true (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6260325?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6260325?buildTab=artifacts#/activerecord) on master @ [770ff3c545a51752490403da64d56fb397f49c5e](https://github.com/cockroachdb/cockroach/commits/770ff3c545a51752490403da64d56fb397f49c5e):


```
		--- FAIL: FinderTest#test_hash_condition_find_with_nil (unexpected)
		--- FAIL: FinderTest#test_find_by_on_attribute_that_is_a_reserved_word (unexpected)
		--- FAIL: FinderTest#test_find_by_with_alias (unexpected)
		--- FAIL: FinderTest#test_exists_with_polymorphic_relation (unexpected)
		--- FAIL: FinderTest#test_take_failing (unexpected)
		--- FAIL: ActiveRecord::ConnectionAdapters::LegacyConnectionHandlersMultiDbTest#test_loading_relations_with_multi_db_connection_handlers (unexpected)
		--- FAIL: AssociationProxyTest#test_proxy_object_is_cached (unexpected)
		--- FAIL: FinderTest#test_finder_with_offset_string (unexpected)
		--- FAIL: ActiveRecord::Migration::CompatibilityTest#test_datetime_doesnt_set_precision_on_change_table_4_2 (unexpected)
		--- FAIL: FinderTest#test_find_by_with_hash_conditions_returns_the_first_matching_record (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnAHasOneAssociation#test_callbacks_firing_order_on_save (unexpected)
		--- FAIL: FinderTest#test_find_only_some_columns (unexpected)
		--- FAIL: FinderTest#test_hash_condition_find_with_array (unexpected)
		--- FAIL: FinderTest#test_find_on_hash_conditions_with_numeric_range_for_string (unexpected)
		--- FAIL: FinderTest#test_implicit_order_column_is_configurable (unexpected)
		--- FAIL: DDLTest#test_create_simple_geometry_using_shortcut (unexpected)
		--- FAIL: FinderTest#test_find_by_empty_ids (unexpected)
		--- FAIL: DefaultStringsTest#test_default_strings (unexpected)
		--- FAIL: DDLTest#test_column_defaults (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: FinderTest#test_exists_with_aggregate_having_three_mappings_with_one_difference (unexpected)
		--- FAIL: FinderTest#test_find_on_hash_conditions_with_array_of_ranges (unexpected)
		--- FAIL: ActiveRecord::InvertibleMigrationTest#test_migrate_down_with_table_name_prefix (unexpected)
		--- FAIL: FinderTest#test_find_with_nil_inside_set_passed_for_attribute (unexpected)
		--- FAIL: FinderTest#test_take (unexpected)
		--- FAIL: ActiveRecord::Migration::CompatibilityTest#test_timestamps_have_null_constraints_if_not_present_in_migration_for_adding_timestamps_to_existing_table (unexpected)
		--- FAIL: DDLTest#test_array_columns (unexpected)
		--- FAIL: FinderTest#test_member_on_unloaded_relation_with_offset (unexpected)
		--- FAIL: FinderTest#test_find_by_and_where_consistency_with_active_record_instance (unexpected)
		--- FAIL: FinderTest#test_find_on_multiple_hash_conditions (unexpected)
		--- FAIL: ActiveRecord::Migration::CompatibilityTest#test_datetime_doesnt_set_precision_on_change_table_6_0 (unexpected)
		--- FAIL: LegacyPrimaryKeyTest::V5_0#test_legacy_join_table_column_options_should_be_overwritten (unexpected)
		--- FAIL: AssociationsExtensionsTest#test_extension_on_habtm (unexpected)
		--- FAIL: FinderTest#test_find_one_message_with_custom_primary_key (unexpected)
		--- FAIL: FinderTest#test_member_on_loaded_relation_with_match (unexpected)
		--- FAIL: AssociationProxyTest#test_reset_unloads_target (unexpected)
		--- FAIL: FinderTest#test_exists_with_aggregate_having_three_mappings (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_change_column_of_parent_table (unexpected)
		--- FAIL: AssociationProxyTest#test_pluck_uses_loaded_target (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: FinderTest#test_second_have_primary_key_order_by_default (unexpected)
		--- FAIL: FinderTest#test_take_and_first_and_last_with_integer_should_use_sql_limit (unexpected)
		--- FAIL: ActiveRecord::CacheKeyTest#test_cache_key_format_is_not_too_precise (unexpected)
		--- FAIL: FinderTest#test_find_on_hash_conditions_with_qualified_attribute_dot_notation_symbol (unexpected)
		--- FAIL: ActiveRecord::ConnectionAdapters::ConnectionHandlersShardingDbTest#test_same_shards_across_clusters (unexpected)
		--- FAIL: DDLTest#test_create_simple_geography (unexpected)
		--- FAIL: FinderTest#test_find_by_with_range_conditions_returns_the_first_matching_record (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6267021?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6267021?buildTab=artifacts#/activerecord) on master @ [f59620ec646d1181d358d0dc41ab60815ecf59c9](https://github.com/cockroachdb/cockroach/commits/f59620ec646d1181d358d0dc41ab60815ecf59c9):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.00000000-3212-gf59620ec64
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6273143?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6273143?buildTab=artifacts#/activerecord) on master @ [3b16435371a43d603d193a1e2b480a23fba3f07a](https://github.com/cockroachdb/cockroach/commits/3b16435371a43d603d193a1e2b480a23fba3f07a):


```
		--- FAIL: DDLTest#test_add_geometry_column_null_false (unexpected)
		--- FAIL: EagerAssociationTest#test_preload_has_many_using_primary_key (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_on_multiple_records (unexpected)
		--- FAIL: EagerAssociationTest#test_polymorphic_type_condition (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_creation_respects_hash_condition (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_limit_and_conditions (unexpected)
		--- FAIL: HasOneThroughAssociationsTest#test_has_one_through_relationship_cannot_have_a_counter_cache (unexpected)
		--- FAIL: OptimisticLockingTest#test_quote_value_passed_lock_col (unexpected)
		--- FAIL: StoreTestWithYAMLSafeLoad#test_all_stored_attributes_are_returned (unexpected)
		--- FAIL: EagerAssociationTest#test_with_two_tables_in_from_without_getting_double_quoted (unexpected)
		--- FAIL: HasOneThroughAssociationsTest#test_association_build_constructor_builds_through_record (unexpected)
		--- FAIL: EagerAssociationTest#test_eager-loading_non-readonly_association (unexpected)
		--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_2#test_polymorphic_add_reference_uses_all_column_names_in_index (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_with_custom_name_counter_cache (unexpected)
		--- FAIL: StoreTestWithYAMLSafeLoad#test_updating_the_store_will_mark_accessor_as_changed (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_belongs_to_without_counter_cache_option (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_add_remove_single_field_using_string_arguments (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_assignment_foreign_type_field_updating (unexpected)
		--- FAIL: InheritanceTest#test_company_descends_from_active_record (unexpected)
		--- FAIL: NestedThroughAssociationsTest#test_has_many_through_has_and_belongs_to_many_with_has_many_source_reflection_preload (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_has_and_belongs_to_many_while_partial_inserts_false (unexpected)
		--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association_with_order_on_nested_association (unexpected)
		--- FAIL: HasOneThroughAssociationsTest#test_has_one_through_many_raises_exception (unexpected)
		--- FAIL: DDLTest#test_non_spatial_column_limits (unexpected)
		--- FAIL: NestedThroughAssociationsTest#test_has_many_through_belongs_to_with_has_many_through_source_reflection_preload_via_joins (unexpected)
		--- FAIL: UpdateAllTest#test_update_all_with_non_standard_table_name (unexpected)
		--- FAIL: EagerAssociationTest#test_associations_loaded_for_all_records (unexpected)
		--- FAIL: StrictLoadingTest#test_strict_loading_by_default (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_attributes_are_being_set_when_initialized_from_habtm_association_with_multiple_where_clauses (unexpected)
		--- FAIL: HasOneThroughAssociationsTest#test_has_one_through_eager_loading (unexpected)
		--- FAIL: EagerAssociationTest#test_base_messages (unexpected)
		--- FAIL: HasManyThroughFixture#test_has_many_through_with_default_table_name_on_join_table (unexpected)
		--- FAIL: DDLTest#test_create_point_geometry_using_shortcut (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_multiple_associations_with_same_table_belongs_to (unexpected)
		--- FAIL: StoreTestWithYAMLSafeLoad#test_read_accessor_without_pre/suffix_in_the_same_store_as_other_pre/suffixed_accessors_still_works (unexpected)
		--- FAIL: StrictLoadingTest#test_raises_on_lazy_loading_a_has_one_relation_if_strict_loading_by_default (unexpected)
		--- FAIL: EagerAssociationTest#test_associations_with_extensions_are_not_instance_dependent (unexpected)
		--- FAIL: EagerAssociationTest#test_eager-loading_with_a_polymorphic_association_won't_work_consistently (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_find_should_append_to_association_order (unexpected)
		--- FAIL: StrictLoadingTest#test_raises_on_unloaded_relation_methods_if_strict_loading_by_default (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_default_scope_as_class_method_using_find_method (unexpected)
		--- FAIL: InheritanceTest#test_alt_inheritance_save (unexpected)
		--- FAIL: InheritanceTest#test_inheritance_find_all (unexpected)
		--- FAIL: DDLTest#test_add_geometry_column (unexpected)
		--- FAIL: InheritanceTest#test_find_first_within_inheritance (unexpected)
		--- FAIL: StrictLoadingTest#test_strict_loading_can_be_turned_off_on_an_association_in_a_model_with_strict_loading_on (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_conditions_hash (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6287501?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6287501?buildTab=artifacts#/activerecord) on master @ [d35c174b71065264b8a3910df3f488d10741f788](https://github.com/cockroachdb/cockroach/commits/d35c174b71065264b8a3910df3f488d10741f788):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.00000000-3264-gd35c174b71
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7358 tests passed
		14 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		14 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_reference_column_of_child_table (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_change_column_of_parent_table (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6315043?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6315043?buildTab=artifacts#/activerecord) on master @ [b316a5ed5fe7253d113174d9d95ddebf1143b4e4](https://github.com/cockroachdb/cockroach/commits/b316a5ed5fe7253d113174d9d95ddebf1143b4e4):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.1-239-gb316a5ed5f
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7363 tests passed
		9 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		9 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_simple_interval (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6329079?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6329079?buildTab=artifacts#/activerecord) on master @ [c080f7ac2f5be11aedd787dceba77cca7df18a16](https://github.com/cockroachdb/cockroach/commits/c080f7ac2f5be11aedd787dceba77cca7df18a16):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.1-293-gc080f7ac2f
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6336049?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6336049?buildTab=artifacts#/activerecord) on master @ [4dcb32c0346e20a95847763f89b9b0796d9ed4dc](https://github.com/cockroachdb/cockroach/commits/4dcb32c0346e20a95847763f89b9b0796d9ed4dc):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.1-303-g4dcb32c034
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6341855?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6341855?buildTab=artifacts#/activerecord) on master @ [b0f13c9bbef3e6628471d887672be7c7658f6511](https://github.com/cockroachdb/cockroach/commits/b0f13c9bbef3e6628471d887672be7c7658f6511):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.1-305-gb0f13c9bbe
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6347309?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6347309?buildTab=artifacts#/activerecord) on master @ [e5917605f751c569ddd90062250a40cafd7ccf30](https://github.com/cockroachdb/cockroach/commits/e5917605f751c569ddd90062250a40cafd7ccf30):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.1-309-ge5917605f7
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6354744?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6354744?buildTab=artifacts#/activerecord) on master @ [6ffaa1d907b7c9194df4ffff412f48a63bf432df](https://github.com/cockroachdb/cockroach/commits/6ffaa1d907b7c9194df4ffff412f48a63bf432df):


```
		--- FAIL: EagerAssociationTest#test_has_many_through_with_order (unexpected)
		--- FAIL: EagerAssociationTest#test_preload_has_many_using_primary_key (unexpected)
		--- FAIL: UpdateAllTest#test_update_all_with_joins_and_limit_and_order (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_load_has_many_through_with_string_keys (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_going_through_join_model (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_unavailable_through_reflection (unexpected)
		--- FAIL: BasePreventWritesTest#test_preventing_writes_applies_to_all_connections_in_block (unexpected)
		--- FAIL: UpdateAllTest#test_update_all_cares_about_optimistic_locking (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_calculations_on_has_many_through_should_disambiguate_fields (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association (unexpected)
		--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_recursive_cascading_four_levels_has_many_through (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_and_limit (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_order_string_with_unquoted_table_name (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_deleting_by_integer_id_from_has_many_through (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- FAIL: UpdateAllTest#test_update_all_with_joins (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_loading_with_conditions_on_string_joined_table_preloads (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_and_belongs_to_many_and_limit (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_and_limit_and_conditions_array (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_through_include_checks_if_record_exists_if_target_not_loaded (unexpected)
		--- FAIL: FixturesTest#test_broken_yaml_exception (unexpected)
		--- FAIL: TestAutosaveAssociationOnABelongsToAssociationDefinedAsRecord#test_should_not_raise_error (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_alternate_database (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_through_an_sti_join_model (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_set_polymorphic_has_one_on_new_record (unexpected)
		--- FAIL: EagerAssociationTest#test_associations_loaded_for_all_records (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_foreign_keys (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_not_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: UpdateAllTest#test_update_all_with_blank_argument (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_belongs_to_polymorphic_with_counter_cache (unexpected)
		--- FAIL: UpdateAllTest#test_update_all_with_joins_and_offset_and_order (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_order_string_with_quoted_table_name (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_distinct_has_many_through_should_retain_order (unexpected)
		--- FAIL: UpdateAllTest#test_update_all_with_group_by (unexpected)
		--- FAIL: UpdateAllTest#test_update_bang_on_relation (unexpected)
		--- FAIL: EagerAssociationTest#test_has_many_association_ignores_the_scoping (unexpected)
		--- FAIL: FixturesTest#test_erb_in_fixtures (unexpected)
		--- FAIL: FixturesTest#test_inserts (unexpected)
		--- FAIL: EagerSingularizationTest#test_eager_no_extra_singularization_has_many_through_belongs_to (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_limit_and_offset_and_conditions (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_and_limit_and_conditions (unexpected)
		--- FAIL: EagerAssociationTest#test_exceptions_have_suggestions_for_fix (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_distinct_through_count (unexpected)
		--- FAIL: FixturesTest#test_nonexistent_fixture_file (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_delete_associate_when_deleting_from_has_many_through (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6369513?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6369513?buildTab=artifacts#/activerecord) on master @ [2372698da1dfacb90f60c6a63f2c1298d1db16b8](https://github.com/cockroachdb/cockroach/commits/2372698da1dfacb90f60c6a63f2c1298d1db16b8):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:897: 
		Tests run on Cockroach v22.2.0-alpha.2-246-g2372698da1
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6383257?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6383257?buildTab=artifacts#/activerecord) on master @ [e39111b2e714375faa0facc05e51e8f619a55b21](https://github.com/cockroachdb/cockroach/commits/e39111b2e714375faa0facc05e51e8f619a55b21):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:908: 
		Tests run on Cockroach v22.2.0-alpha.2-305-ge39111b2e7
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7358 tests passed
		14 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		14 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_column_of_child_table (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_change_column_of_parent_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6400376?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6400376?buildTab=artifacts#/activerecord) on master @ [a82711442c65cf14489c55041b45b11a1e38415b](https://github.com/cockroachdb/cockroach/commits/a82711442c65cf14489c55041b45b11a1e38415b):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:251,test_runner.go:906: 
		Tests run on Cockroach v22.2.0-alpha.2-371-ga82711442c
		Tests run against activerecord 7.0.3
		7372 Total Tests Run
		7364 tests passed
		8 tests failed
		27 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		8 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6415593?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6415593?buildTab=artifacts#/activerecord) on master @ [389661e823c19f318fa07ec2278336262531692d](https://github.com/cockroachdb/cockroach/commits/389661e823c19f318fa07ec2278336262531692d):


```
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: PersistenceTest#test_class_level_destroy_is_affected_by_scoping (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_constraint_by_name (unexpected)
		--- FAIL: BasicsTest#test_create_without_prepared_statement (unexpected)
		--- FAIL: PersistenceTest#test_delete_all (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: BasicsTest#test_marshal_new_record_round_trip (unexpected)
		--- FAIL: PersistenceTest#test_update_columns_should_not_leave_the_object_dirty (unexpected)
		--- FAIL: BasicsTest#test_column_types_typecast (unexpected)
		--- FAIL: PersistenceTest#test_update_many_with_array_of_active_record_base_objects (unexpected)
		--- FAIL: HasManyThroughFixture#test_has_many_through_with_join_table_name_changed_to_match_habtm_table_name (unexpected)
		--- FAIL: BasicsTest#test_find_on_abstract_base_class_doesnt_use_type_condition (unexpected)
		--- FAIL: BasicsTest#test_default_in_local_time (unexpected)
		--- FAIL: PersistenceTest#test_increment_updates_counter_in_db_using_offset (unexpected)
		--- FAIL: PersistenceTest#test_update_attribute (unexpected)
		--- FAIL: PersistenceTest#test_destroyed_returns_boolean (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_name (unexpected)
		--- FAIL: PersistenceTest#test_decrement_attribute (unexpected)
		--- FAIL: BasicsTest#test_marshal_round_trip (unexpected)
		--- FAIL: BasicsTest#test_has_attribute_with_symbol (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: BasicsTest#test_when_ignored_attribute_is_loaded,_cast_type_should_be_preferred_over_DB_type (unexpected)
		--- FAIL: PersistenceTest#test_update_columns_should_not_use_setter_method (unexpected)
		--- FAIL: BasicsTest#test_auto_id (unexpected)
		--- FAIL: BasicsTest#test_create_after_initialize_with_block (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
		--- FAIL: PersistenceTest#test_update_columns_should_not_modify_updated_at (unexpected)
		--- FAIL: PersistenceTest#test_decrement_with_touch_an_attribute_updates_timestamps (unexpected)
		--- FAIL: BasicsTest#test_dup (unexpected)
		--- FAIL: BasicsTest#test_all_with_conditions (unexpected)
		--- FAIL: BasicsTest#test_load (unexpected)
		--- FAIL: BasicsTest#test_dont_clear_inheritance_column_when_setting_explicitly (unexpected)
		--- FAIL: PersistenceTest#test_create_through_factory_with_block (unexpected)
		--- FAIL: BasicsTest#test_cannot_call_connected_to_on_subclasses_of_ActiveRecord::Base_with_legacy_connection_handling (unexpected)
		--- FAIL: BasicsTest#test_destroy_without_prepared_statement (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_prefix (unexpected)
		--- FAIL: BasicsTest#test_table_name_guesses_with_prefixes_and_suffixes (unexpected)
		--- FAIL: PersistenceTest#test_reload_via_querycache (unexpected)
		--- FAIL: PersistenceTest#test_update_column_with_model_having_primary_key_other_than_id (unexpected)
		--- FAIL: PersistenceTest#test_update_columns_changing_id (unexpected)
		--- FAIL: PersistenceTest#test_class_level_update_is_affected_by_scoping (unexpected)
		--- FAIL: PersistenceTest#test_destroy_many_with_invalid_id (unexpected)
		--- FAIL: PersistenceTest#test_update_column_should_raise_exception_if_new_record (unexpected)
		--- FAIL: BasicsTest#test_new_threads_get_default_the_default_connection_handler (unexpected)
		--- FAIL: BasicsTest#test_values_at (unexpected)
		--- FAIL: BasicsTest#test_find_by_slug_with_array (unexpected)
		--- FAIL: PersistenceTest#test_delete_new_record (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6423857?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6423857?buildTab=artifacts#/activerecord) on master @ [bc2e47da0523b347c28cf024707e80cd35d6c98a](https://github.com/cockroachdb/cockroach/commits/bc2e47da0523b347c28cf024707e80cd35d6c98a):


```
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_infers_column (unexpected)
		--- FAIL: HasManyAssociationsTest#test_collection_size_after_building (unexpected)
		--- FAIL: HasManyAssociationsTest#test_abstract_class_with_polymorphic_has_many (unexpected)
		--- FAIL: EagerSingularizationTest#test_eager_no_extra_singularization_belongs_to (unexpected)
		--- FAIL: OptimisticLockingTest#test_lock_existing (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dependence_with_transaction_support_on_failure (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_with_piggyback (unexpected)
		--- FAIL: LegacyPrimaryKeyTest::V5_0#test_legacy_join_table_column_options_should_be_overwritten (unexpected)
		--- FAIL: HasManyAssociationsTest#test_adding_a_collection (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: AssociationProxyTest#test_size_differentiates_between_new_and_persisted_in_memory_records_when_loaded_records_are_empty (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_going_through_join_model_with_custom_foreign_key (unexpected)
		--- FAIL: ActiveRecord::Migration::CompatibilityTest#test_datetime_doesnt_set_precision_on_change_table_6_1 (unexpected)
		--- FAIL: HasManyAssociationsTest#test_finding_with_different_class_name_and_order (unexpected)
		--- FAIL: HasManyAssociationsTest#test_delete_all_with_not_yet_loaded_association_collection (unexpected)
		--- FAIL: RequiredAssociationsTest#test_required_belongs_to_associations_have_presence_validated (unexpected)
		--- FAIL: HotCompatibilityTest#test_cleans_up_after_prepared_statement_failure_in_a_transaction (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_array_methods_called_by_method_missing (unexpected)
		--- FAIL: OptimisticLockingTest#test_lock_without_default_should_work_with_null_in_the_database (unexpected)
		--- FAIL: ActiveRecord::Migration::CompatibilityTest#test_change_column_comment_can_be_reverted (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: OptimisticLockingTest#test_counter_cache_with_touch_and_lock_version (unexpected)
		--- FAIL: HasManyAssociationsTest#test_delete_all,_when_not_loaded,_doesn't_load_the_records (unexpected)
		--- FAIL: HasManyAssociationsTest#test_clearing_an_association_collection (unexpected)
		--- FAIL: HasManyAssociationsTest#test_destroy_does_not_raise_when_association_errors_on_destroy (unexpected)
		--- FAIL: AssociationProxyTest#test_proxy_association_accessor (unexpected)
		--- FAIL: HasManyAssociationsTest#test_first_or_create!_adds_the_record_to_the_association (unexpected)
		--- FAIL: HasManyAssociationsTest#test_include_method_in_has_many_association_should_return_true_for_instance_added_with_build (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dependence_on_account (unexpected)
		--- FAIL: SpatialQueriesTest#test_query_point_wkt (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_deleting_by_string_id_from_has_many_through (unexpected)
		--- FAIL: HasManyAssociationsTest#test_with_polymorphic_has_many_with_custom_columns_name (unexpected)
		--- FAIL: HasManyAssociationsTest#test_create_with_bang_on_has_many_when_parent_is_new_raises (unexpected)
		--- FAIL: HasManyAssociationsTest#test_defining_has_many_association_with_nullify_dependency_lazily_evaluates_target_class (unexpected)
		--- FAIL: AssociationProxyTest#test_inspect_does_not_reload_a_not_yet_loaded_target (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many (unexpected)
		--- FAIL: HasManyAssociationsTest#test_destroy_all_on_desynced_counter_cache_association (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_scoped_grouped_having (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_create_model_with_inheritance_and_custom_base_class (unexpected)
		--- FAIL: LegacyPrimaryKeyTest::V4_2#test_legacy_join_table_column_options_should_be_overwritten (unexpected)
		--- FAIL: HasManyAssociationsTest#test_delete_all (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_through_include_checks_if_record_exists_if_target_not_loaded (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_preload_polymorphic_has_many (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_restrict_foreign_key (unexpected)
		--- FAIL: OptimisticLockingTest#test_lock_column_name_existing (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_going_through_join_model_with_custom_select_and_joins (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_name (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6433129?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6433129?buildTab=artifacts#/activerecord) on master @ [773568fbda06ba9be9fb1bc34a331f21c8891ffa](https://github.com/cockroachdb/cockroach/commits/773568fbda06ba9be9fb1bc34a331f21c8891ffa):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.2-439-g773568fbda
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7334 tests passed
		13 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		13 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6448808?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6448808?buildTab=artifacts#/activerecord) on master @ [2cfa6b1779ead01508c04e228bf72b4e0e96d98c](https://github.com/cockroachdb/cockroach/commits/2cfa6b1779ead01508c04e228bf72b4e0e96d98c):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-228-g2cfa6b1779
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7334 tests passed
		13 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		13 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6471013?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6471013?buildTab=artifacts#/activerecord) on master @ [1f24e52aea4195ffe26b97160079bb29f264338a](https://github.com/cockroachdb/cockroach/commits/1f24e52aea4195ffe26b97160079bb29f264338a):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-264-g1f24e52aea
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7334 tests passed
		13 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		13 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6482566?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6482566?buildTab=artifacts#/activerecord) on master @ [9a05046ce19e7678340e82c70d61e928be95bc72](https://github.com/cockroachdb/cockroach/commits/9a05046ce19e7678340e82c70d61e928be95bc72):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-291-g9a05046ce1
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7334 tests passed
		13 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		13 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6495696?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6495696?buildTab=artifacts#/activerecord) on master @ [bd97ad5b8c9f537a89492a051574d867469bef33](https://github.com/cockroachdb/cockroach/commits/bd97ad5b8c9f537a89492a051574d867469bef33):


```
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_prefix (unexpected)
		--- FAIL: ActiveRecord::CacheKeyTest#test_cache_version_does_call_updated_at_when_it_is_assigned_via_a_string (unexpected)
		--- FAIL: PrimaryKeysTest#test_update_with_non_primary_key_id_column (unexpected)
		--- FAIL: ActiveRecord::CacheKeyTest#test_cache_key_with_version_always_has_both_key_and_version (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping_on_delete_and_on_update_options (unexpected)
		--- FAIL: PrimaryKeysTest#test_customized_primary_key_auto_assigns_on_save (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: PrimaryKeysTest#test_to_key_with_default_primary_key (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_suffix (unexpected)
		--- FAIL: FixturesTest#test_instantiation (unexpected)
		--- FAIL: PrimaryKeysTest#test_create_without_primary_key_no_extra_query (unexpected)
		--- FAIL: FixturesTest#test_subsubdir_file_with_arbitrary_name (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CacheKeyTest#test_cache_version_calls_updated_at_when_the_value_is_generated_at_create_time (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_on_update (unexpected)
		--- FAIL: FixturesTest#test_nil_raises (unexpected)
		--- FAIL: SerializedAttributeTestWithYamlSafeLoad#test_regression_serialized_default_on_text_column_with_null_false (unexpected)
		--- FAIL: DefaultNumbersTest#test_default_negative_integer (unexpected)
		--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyInCreateTest#test_options_hash_can_be_passed (unexpected)
		--- FAIL: CockroachDB::PrimaryKeysTest#test_serial_with_unquoted_sequence_name (unexpected)
		--- FAIL: PrimaryKeysTest#test_quoted_primary_key_after_set_primary_key (unexpected)
		--- FAIL: FixturesTest#test_erb_in_fixtures (unexpected)
		--- FAIL: FixturesTest#test_yml_file_in_subdirectory (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_is_reversible (unexpected)
		--- FAIL: PrimaryKeysTest#test_find_with_multiple_ids_should_quote_pkey (unexpected)
		--- FAIL: FixturesTest#test_fixtures_are_set_up_with_database_env_variable (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: PrimaryKeysTest#test_auto_detect_primary_key_from_schema (unexpected)
		--- FAIL: PrimaryKeysTest#test_primary_key_returns_value_if_it_exists (unexpected)
		--- FAIL: FixturesTest#test_no_args_returns_all (unexpected)
		--- FAIL: PrimaryKeysTest#test_read_attribute_with_custom_primary_key (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_inferes_column (unexpected)
		--- FAIL: FixturesTest#test_empty_yaml_fixture (unexpected)
		--- FAIL: PrimaryKeysTest#test_string_key (unexpected)
		--- FAIL: DefaultNumbersTest#test_default_positive_integer (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_name (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: SerializedAttributeTest#test_regression_serialized_default_on_text_column_with_null_false (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_the_select_one_on_the_same_table (unexpected)
		--- FAIL: FixturesTest#test_logger_level_invariant (unexpected)
		--- FAIL: FixturesTest#test_broken_yaml_exception (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_nullify_foreign_key (unexpected)
		--- FAIL: CockroachDB::PrimaryKeysTest#test_serial_with_quoted_sequence_name (unexpected)
		--- FAIL: PrimaryKeysTest#test_to_key_with_primary_key_after_destroy (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_name (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6504163?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6504163?buildTab=artifacts#/activerecord) on master @ [3fa1a1600898d7b78b9e39d07132a387a2f9a1b6](https://github.com/cockroachdb/cockroach/commits/3fa1a1600898d7b78b9e39d07132a387a2f9a1b6):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-315-g3fa1a16008
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7333 tests passed
		14 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		14 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_simple_interval (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6513239?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6513239?buildTab=artifacts#/activerecord) on master @ [87ed064dc23eab6948ee8a07e8507f150bda0e44](https://github.com/cockroachdb/cockroach/commits/87ed064dc23eab6948ee8a07e8507f150bda0e44):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:917: 
		Tests run on Cockroach v22.2.0-alpha.3-323-g87ed064dc2
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7335 tests passed
		12 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		12 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6530764?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6530764?buildTab=artifacts#/activerecord) on master @ [5862e9b3c1e24b3f644716789c96108a8e92fd71](https://github.com/cockroachdb/cockroach/commits/5862e9b3c1e24b3f644716789c96108a8e92fd71):


```
		--- FAIL: FinderTest#test_fifth (unexpected)
		--- FAIL: ReflectionTest#test_association_primary_key_raises_when_missing_primary_key (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_many_should_return_true_if_more_than_one (unexpected)
		--- FAIL: HasManyAssociationsTest#test_deleting_before_save (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_update_counter_caches_on_delete_with_dependent_nullify (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_delete_on_association_clears_scope (unexpected)
		--- FAIL: FinderTest#test_last_on_relation_with_limit_and_offset (unexpected)
		--- FAIL: FinderTest#test_second_with_offset (unexpected)
		--- FAIL: HasManyAssociationsTest#test_destroy_does_not_raise_when_association_errors_on_destroy (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_through_join_model_with_conditions_on_top_level (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_update_changing_ids_of_inversed_association_changes_the_counter_cache (unexpected)
		--- FAIL: FinderTest#test_eager_load_for_no_has_many_with_limit_and_joins_for_has_many (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_has_many_association_through_a_belongs_to_association (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_add_two_instance_and_then_deleting (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_with_piggyback (unexpected)
		--- FAIL: EagerAssociationTest#test_deep_including_through_habtm (unexpected)
		--- FAIL: MultipleDbTest#test_course_connection_should_survive_reloads (unexpected)
		--- FAIL: EagerAssociationTest#test_including_duplicate_objects_from_has_many (unexpected)
		--- FAIL: RelationTest#test_arel_table_respects_a_custom_table (unexpected)
		--- FAIL: FinderTest#test_#skip_query_cache!_for_#exists? (unexpected)
		--- FAIL: HasOneAssociationsTest#test_create_with_block (unexpected)
		--- FAIL: FinderTest#test_third_to_last (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_through_with_conditions_join_model_with_include (unexpected)
		--- FAIL: RelationTest#test_create (unexpected)
		--- FAIL: CounterCacheTest#test_decrement_counter (unexpected)
		--- FAIL: PrimaryKeysTest#test_primary_key_returns_nil_if_it_does_not_exist (unexpected)
		--- FAIL: RelationTest#test_find_ids (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_user_provided_defaults_are_persisted_even_if_unchanged (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_belongs_to_shared_parent (unexpected)
		--- FAIL: OptimisticLockingTest#test_explicit_update_lock_column_raise_error (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_can_update_through_association (unexpected)
		--- FAIL: NestedThroughAssociationsTest#test_has_many_through_has_many_with_has_and_belongs_to_many_source_reflection (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_with_table_name (unexpected)
		--- FAIL: RelationTest#test_distinct (unexpected)
		--- FAIL: FinderTest#test_last_with_integer_and_order_should_keep_the_order (unexpected)
		--- FAIL: HasManyAssociationsTest#test_clearing_a_dependent_association_collection (unexpected)
		--- FAIL: HasManyAssociationsTest#test_has_many_through_respects_hash_conditions (unexpected)
		--- FAIL: FinderTest#test_include_on_unloaded_relation_without_match (unexpected)
		--- FAIL: FinderTest#test_find_with_ids_and_limit (unexpected)
		--- FAIL: AssociationsJoinModelTest#test_has_many_inherited (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: BasicsTest#test_primary_key_with_no_id (unexpected)
		--- FAIL: EagerAssociationTest#test_preloading_associations_with_string_joins_and_order_references (unexpected)
		--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_has_one_reflection (unexpected)
		--- FAIL: EagerAssociationTest#test_limited_eager_with_numeric_in_association (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_with_decimal_for_string_column (unexpected)
		--- FAIL: RelationTest#test_references_with_blank_value (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6549169?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6549169?buildTab=artifacts#/activerecord) on master @ [bbcb10f982e47943de63a3a2d1faf10d3eb49e75](https://github.com/cockroachdb/cockroach/commits/bbcb10f982e47943de63a3a2d1faf10d3eb49e75):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-430-gbbcb10f982
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7334 tests passed
		13 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		13 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6567232?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6567232?buildTab=artifacts#/activerecord) on master @ [726cf22b9f06b766d857b4617dec0df18d1e5cd0](https://github.com/cockroachdb/cockroach/commits/726cf22b9f06b766d857b4617dec0df18d1e5cd0):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-472-g726cf22b9f
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7334 tests passed
		13 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		13 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6584253?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6584253?buildTab=artifacts#/activerecord) on master @ [34dc56fbb5789b39be47b110bf22332c7f5654f6](https://github.com/cockroachdb/cockroach/commits/34dc56fbb5789b39be47b110bf22332c7f5654f6):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-527-g34dc56fbb5
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7333 tests passed
		14 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		14 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_simple_interval (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6601244?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6601244?buildTab=artifacts#/activerecord) on master @ [89f4ad907a1756551bd6864c3e8516eeff6b0e0a](https://github.com/cockroachdb/cockroach/commits/89f4ad907a1756551bd6864c3e8516eeff6b0e0a):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-581-g89f4ad907a
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7333 tests passed
		14 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		14 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_simple_interval (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6610307?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6610307?buildTab=artifacts#/activerecord) on master @ [51c8aae748d338549400c047796c6c9b892527da](https://github.com/cockroachdb/cockroach/commits/51c8aae748d338549400c047796c6c9b892527da):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-583-g51c8aae748
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7334 tests passed
		13 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		13 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6619262?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6619262?buildTab=artifacts#/activerecord) on master @ [ff13325e9368c4e8dd9a4d5cf4aa2ad2f33e9ac0](https://github.com/cockroachdb/cockroach/commits/ff13325e9368c4e8dd9a4d5cf4aa2ad2f33e9ac0):


```
		0 tests passed unexpectedly
		40 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: MigrationTest#test_internal_metadata_table_name (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_on_update (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_suffix (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_is_reversible (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: CockroachDB::PostgresqlTimestampMigrationTest#test_adds_column_as_custom_type (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_with_if_exists_not_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_symbol_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: CockroachDB::PostgresqlTimestampMigrationTest#test_adds_column_as_timestamptz_if_datetime_type_changed (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_invalid_foreign_key (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_cascade_foreign_key (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- FAIL: CockroachDB::PostgresqlTimestampMigrationTest#test_adds_column_as_timestamp (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_inferes_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_constraint_by_name (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_change_column_of_parent_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping_on_delete_and_on_update_options (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_with_if_exists_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_prefix (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_inferes_column (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6628566?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6628566?buildTab=artifacts#/activerecord) on master @ [ff13325e9368c4e8dd9a4d5cf4aa2ad2f33e9ac0](https://github.com/cockroachdb/cockroach/commits/ff13325e9368c4e8dd9a4d5cf4aa2ad2f33e9ac0):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-593-gff13325e93
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7333 tests passed
		14 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		14 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_simple_interval (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6638330?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6638330?buildTab=artifacts#/activerecord) on master @ [a0bfa6dafcc206301d3a21887c374db63b377075](https://github.com/cockroachdb/cockroach/commits/a0bfa6dafcc206301d3a21887c374db63b377075):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-643-ga0bfa6dafc
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7334 tests passed
		13 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		13 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6661603?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6661603?buildTab=artifacts#/activerecord) on master @ [4982322e91c37c6d0249dc5597b9e02eec19848d](https://github.com/cockroachdb/cockroach/commits/4982322e91c37c6d0249dc5597b9e02eec19848d):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-690-g4982322e91
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7335 tests passed
		12 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		12 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6683300?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6683300?buildTab=artifacts#/activerecord) on master @ [4780ee15194e7d22b661ac92254b13cd2b71dcb1](https://github.com/cockroachdb/cockroach/commits/4780ee15194e7d22b661ac92254b13cd2b71dcb1):


```
		--- FAIL: InverseBelongsToTests#test_child_instance_should_be_shared_with_replaced_via_accessor_parent (unexpected)
		--- FAIL: HasManyAssociationsTest#test_destroy_all (unexpected)
		--- FAIL: ActiveRecord::DelegationRelationTest#test_delegates_index_to_Array (unexpected)
		--- FAIL: ActiveRecord::Migration::CommandRecorderTest#test_invert_add_index_with_algorithm_option (unexpected)
		--- FAIL: FinderRespondToTest#test_should_preserve_normal_respond_to_behaviour_and_respond_to_newly_added_method (unexpected)
		--- FAIL: TestAutosaveAssociationOnABelongsToAssociation#test_should_automatically_save_bang_the_associated_model (unexpected)
		--- FAIL: ActiveRecord::Encryption::EncryptableRecordTest#test_reading_a_not_encrypted_value_won't_raise_a_Decryption_error_when_:support_unencrypted_data_is_true (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_none_should_return_true_if_none (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnsTest#test_rename_column_using_symbol_arguments (unexpected)
		--- FAIL: YamlSerializationTest#test_cast_types_are_not_changed_on_round_trip (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_where_with_decimal_for_string_column (unexpected)
		--- FAIL: HasManyAssociationsTest#test_deleting_updates_counter_cache_with_dependent_delete_all (unexpected)
		--- FAIL: CalculationsTest#test_pluck (unexpected)
		--- FAIL: BasePreventWritesTest#test_an_empty_transaction_does_not_raise_if_preventing_writes (unexpected)
		--- FAIL: AttributeMethodsTest#test_write_attribute_allows_writing_to_aliased_attributes (unexpected)
		--- FAIL: QueryLogsTest#test_empty_comments_are_not_added (unexpected)
		--- FAIL: DatabaseConfigurationsTest#test_empty_returns_true_when_db_configs_are_empty (unexpected)
		--- FAIL: CopyMigrationsTest#test_copying_migrations_with_timestamps (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_join_table_composite_primary_key_should_not_warn (unexpected)
		--- FAIL: CalculationsTest#test_pluck_loaded_relation_multiple_columns (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_through_join_model_ignores_default_includes (unexpected)
		--- FAIL: DDLTest#test_create_simple_geometry (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_many_on_loaded_association_should_not_use_query (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_association_with_extend_option (unexpected)
		--- FAIL: ActiveRecord::ConnectionAdapters::LegacyConnectionHandlersShardingDbTest#test_establish_connection_using_3_levels_config_with_shards_and_replica (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_where_on_polymorphic_association_with_nil (unexpected)
		--- FAIL: SecureTokenTest#test_regenerating_the_secure_token (unexpected)
		--- FAIL: AssociationValidationTest#test_validates_associated_one (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_default_scope_as_class_method_using_find_method (unexpected)
		--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_validation_does_not_validate_non_dirty_association_target (unexpected)
		--- FAIL: ActiveRecord::WhereTest#test_decorated_polymorphic_where (unexpected)
		--- FAIL: TestAutosaveAssociationsInGeneral#test_autosave_works_even_when_other_callbacks_update_the_parent_model (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_count (unexpected)
		--- FAIL: ActiveRecord::BindParameterTest#test_statement_cache_with_find_by (unexpected)
		--- FAIL: ActiveRecord::LoadAsyncNullExecutorTest#test_scheduled? (unexpected)
		--- FAIL: MultipleDbTest#test_connection (unexpected)
		--- FAIL: HasManyAssociationsTest#test_updates_counter_cache_when_default_scope_is_given (unexpected)
		--- FAIL: HasManyAssociationsTest#test_counter_cache_updates_in_memory_after_update_with_inverse_of_enabled (unexpected)
		--- FAIL: DDLTest#test_create_point_geometry (unexpected)
		--- FAIL: HasManyAssociationsTest#test_build_many_via_block (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_id_assignment (unexpected)
		--- FAIL: HasAndBelongsToManyAssociationsTest#test_habtm_collection_size_from_params (unexpected)
		--- FAIL: HasManyAssociationsTest#test_delete_all_with_not_yet_loaded_association_collection (unexpected)
		--- FAIL: CalculationsTest#test_should_count_selected_field_with_include (unexpected)
		--- FAIL: ActiveRecord::ConnectionAdapters::LegacyConnectionHandlersShardingDbTest#test_connects_to_raises_with_a_shard_and_database_key (unexpected)
		--- FAIL: TestNestedAttributesInGeneral#test_should_not_build_a_new_record_if_reject_all_blank_returns_false (unexpected)
		--- FAIL: ActiveRecord::DelegationRelationTest#test_delegates_&_to_Array (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6698314?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6698314?buildTab=artifacts#/activerecord) on master @ [4780ee15194e7d22b661ac92254b13cd2b71dcb1](https://github.com/cockroachdb/cockroach/commits/4780ee15194e7d22b661ac92254b13cd2b71dcb1):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-734-g4780ee1519
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7334 tests passed
		13 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		13 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6704781?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6704781?buildTab=artifacts#/activerecord) on master @ [801bfc62afd7128be180e3396d21a1e0b2daa227](https://github.com/cockroachdb/cockroach/commits/801bfc62afd7128be180e3396d21a1e0b2daa227):


```
		--- FAIL: ActiveRecord::Migration::ColumnsTest#test_change_column_with_nil_default (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_doesn't_trigger_any_query_if_the_relation_is_already_loaded (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_custom_timestamp_column (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_table_alias (unexpected)
		--- FAIL: ActiveRecord::ConnectionAdapters::SchemaCacheTest#test_yaml_dump_and_load (unexpected)
		--- FAIL: ActiveRecord::ConnectionAdapters::SchemaCacheTest#test_marshal_dump_and_load (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnsTest#test_change_column_nullability (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_version_for_relation (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnsTest#test_change_type_of_not_null_column (unexpected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_simple_interval (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_with_includes_will_update_cache_key (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_relation (unexpected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_schema_dump_with_default_value (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_should_be_stable_when_using_collection_cache_versioning (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_loaded_collection_with_zero_size (unexpected)
		--- FAIL: LegacyPolymorphicReferenceIndexTest::V4_2#test_polymorphic_add_reference_uses_all_column_names_in_index (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_custom_select_and_limit (unexpected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_postgresql_interval_parser (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_unknown_timestamp_column (unexpected)
		--- FAIL: RequiredAssociationsTest#test_required_has_one_associations_have_presence_validated (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_it_triggers_at_most_one_query (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::ConnectionAdapters::SchemaCacheTest#test_marshal_dump_and_load_with_gzip (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_will_update_cache_key (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: LegacyPolymorphicReferenceIndexTest::V4_2#test_create_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_update_all_will_update_cache_key (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnsTest#test_change_column_to_drop_default_with_null_false (unexpected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_postgresql_interval_parser_error (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation (unexpected)
		--- FAIL: LegacyPolymorphicReferenceIndexTest::V4_2#test_create_join_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
		--- FAIL: ActiveRecord::ConnectionAdapters::SchemaCacheTest#test_marshal_dump_and_load_with_ignored_tables (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_destroy_all_will_update_cache_key (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnsTest#test_rename_column (unexpected)
		--- FAIL: RequiredAssociationsTest#test_required_belongs_to_associations_have_a_correct_error_message (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_delete_all_with_includes_will_update_cache_key (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: CockroachDB::PostgresqlIntervalTest#test_column (unexpected)
		--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_collection_proxy_provides_a_cache_key (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6731749?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6731749?buildTab=artifacts#/activerecord) on master @ [e786cba2b137a671d3846cf7a33e7b9dea2854e6](https://github.com/cockroachdb/cockroach/commits/e786cba2b137a671d3846cf7a33e7b9dea2854e6):


```
		--- FAIL: HasManyAssociationsTest#test_calling_many_should_return_false_if_none_or_one (unexpected)
		--- FAIL: TimestampTest#test_changing_parent_of_a_record_touches_both_new_and_old_parent_record (unexpected)
		--- FAIL: HasManyAssociationsTest#test_deleting_updates_counter_cache_with_dependent_delete_all (unexpected)
		--- FAIL: DDLTest#test_drop_geography_column (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_through_an_sti_join_model (unexpected)
		--- FAIL: SerializedAttributeTestWithYamlSafeLoad#test_serialized_attribute (unexpected)
		--- FAIL: InheritanceTest#test_where_new_with_unrelated_type (unexpected)
		--- FAIL: BasicsTest#test_table_name_guesses_with_inherited_prefixes_and_suffixes (unexpected)
		--- FAIL: SerializedAttributeTest#test_serialized_boolean_value_false (unexpected)
		--- FAIL: BasicsTest#test_table_name_guesses (unexpected)
		--- FAIL: EagerAssociationTest#test_eager-loading_non-readonly_association (unexpected)
		--- FAIL: ActiveRecord::ConnectionAdapters::LegacyConnectionHandlersShardingDbTest#test_sharding_separation (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_first_sanitized (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_polymorphic_counter_cache (unexpected)
		--- FAIL: AssociationProxyTest#test_reset_unloads_target (unexpected)
		--- FAIL: StrictLoadingTest#test_strict_loading_with_has_many_through_cascade_down_to_middle_records (unexpected)
		--- FAIL: HasManyAssociationsTest#test_can_rewhere_the_default_scope_of_the_associated_model (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_natural_assignment_with_primary_key (unexpected)
		--- FAIL: StrictLoadingTest#test_raises_on_lazy_loading_a_belongs_to_relation_if_strict_loading_by_default (unexpected)
		--- FAIL: BasicsTest#test_columns_should_obey_set_primary_key (unexpected)
		--- FAIL: FixturesTest#test_binary_in_fixtures (unexpected)
		--- FAIL: StrictLoadingTest#test_raises_on_lazy_loading_a_strict_loading_belongs_to_relation (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dependent_association_respects_optional_conditions_on_delete (unexpected)
		--- FAIL: BasicsTest#test_arel_attribute_normalization (unexpected)
		--- FAIL: HasOneAssociationsTest#test_creation_failure_due_to_new_record_should_raise_error (unexpected)
		--- FAIL: HasManyAssociationsTest#test_calling_many_should_return_true_if_more_than_one (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_should_set_foreign_key_on_create_association! (unexpected)
		--- FAIL: PersistenceTest#test_update_column_for_readonly_attribute (unexpected)
		--- FAIL: HasManyAssociationsTest#test_included_in_collection (unexpected)
		--- FAIL: BasicsTest#test_connection_handler_can_be_overridden (unexpected)
		--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_has_many_through (unexpected)
		--- FAIL: EagerAssociationTest#test_preloading_a_through_association_twice_does_not_reset_it (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: HasManyAssociationsTest#test_associations_replace_in_memory_when_records_have_the_same_id (unexpected)
		--- FAIL: HasManyAssociationsTest#test_collection_size_after_building (unexpected)
		--- FAIL: SerializedAttributeTest#test_where_by_serialized_attribute_with_hash_in_array (unexpected)
		--- FAIL: PrimaryKeysTest#test_primary_key_prefix (unexpected)
		--- FAIL: HasOneAssociationsTest#test_assignment_before_child_saved (unexpected)
		--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_limit (unexpected)
		--- FAIL: StrictLoadingTest#test_does_not_raise_on_eager_loading_a_strict_loading_has_one_relation (unexpected)
		--- FAIL: BelongsToAssociationsTest#test_natural_assignment_to_nil (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_with_valid_association_as_string_not_symbol (unexpected)
		--- FAIL: EagerAssociationTest#test_eager_loaded_has_one_association_without_primary_key (unexpected)
		--- FAIL: HasManyAssociationsTest#test_included_in_collection_for_new_records (unexpected)
		--- FAIL: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (unexpected)
		--- FAIL: HasManyAssociationsTest#test_modifying_a_through_a_has_many_should_raise (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6741071?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6741071?buildTab=artifacts#/activerecord) on master @ [ff6961ef1741766b5cbe0b64ebf754e82343eb00](https://github.com/cockroachdb/cockroach/commits/ff6961ef1741766b5cbe0b64ebf754e82343eb00):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-829-gff6961ef17
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7328 tests passed
		19 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		19 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_rename_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_reference_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_foreign_key_by_column (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_change_column_of_parent_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_rename_column_of_child_table (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6749797?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6749797?buildTab=artifacts#/activerecord) on master @ [84384b50c023dd4c05fff76af85a6975f5d2b0ab](https://github.com/cockroachdb/cockroach/commits/84384b50c023dd4c05fff76af85a6975f5d2b0ab):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-835-g84384b50c0
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7334 tests passed
		13 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		13 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6767310?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6767310?buildTab=artifacts#/activerecord) on master @ [0eaeeb773474716753781289788fdd087fb9b166](https://github.com/cockroachdb/cockroach/commits/0eaeeb773474716753781289788fdd087fb9b166):


```
		--- FAIL: HasOneAssociationsTest#test_cant_save_readonly_association (unexpected)
		--- FAIL: RelationTest#test_finding_with_subquery (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_all_with_include_and_conditions (unexpected)
		--- FAIL: HasManyAssociationsTest#test_find_scoped_grouped (unexpected)
		--- FAIL: PostgresqlBitStringTest#test_bit_string_varying_column (unexpected)
		--- FAIL: HasManyAssociationsTest#test_create_resets_cached_counters (unexpected)
		--- FAIL: HasManyAssociationsTest#test_assign_ids_ignoring_blanks (unexpected)
		--- FAIL: FixturesTest#test_nil_raises (unexpected)
		--- FAIL: RelationTest#test_many_with_limits (unexpected)
		--- FAIL: HasManyAssociationsTest#test_delete_all_association_with_primary_key_deletes_correct_records (unexpected)
		--- FAIL: HasManyAssociationsTest#test_taking (unexpected)
		--- FAIL: HasManyAssociationsTest#test_counting_with_counter_sql (unexpected)
		--- FAIL: RelationTest#test_dynamic_finder (unexpected)
		--- FAIL: HasManyAssociationsTest#test_finding_default_orders (unexpected)
		--- FAIL: RelationTest#test_empty_with_zero_limit (unexpected)
		--- FAIL: HasManyAssociationsTest#test_collection_size_with_dirty_target (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dont_call_save_callbacks_twice_on_has_many (unexpected)
		--- FAIL: RelationTest#test_first_or_create_bang_with_valid_options (unexpected)
		--- FAIL: RelationTest#test_count_with_block (unexpected)
		--- FAIL: RelationTest#test_anonymous_extension (unexpected)
		--- FAIL: HasManyAssociationsTest#test_prevent_double_insertion_of_new_object_when_the_parent_association_loaded_in_the_after_save_callback (unexpected)
		--- FAIL: HasManyAssociationsTest#test_to_a_should_dup_target (unexpected)
		--- FAIL: HasOneAssociationsTest#test_has_one (unexpected)
		--- FAIL: HasManyAssociationsTest#test_counter_cache_updates_in_memory_after_create_with_array (unexpected)
		--- FAIL: FixturesTest#test_broken_yaml_exception (unexpected)
		--- FAIL: RelationTest#test_pluck_with_from_includes_quoted_original_table_name (unexpected)
		--- FAIL: HasOneAssociationsTest#test_build (unexpected)
		--- FAIL: RelationTest#test_to_xml (unexpected)
		--- FAIL: RelationTest#test_loaded_first_with_limit (unexpected)
		--- FAIL: RelationTest#test_find_all_using_where_with_relation_with_joins (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_model_with_nonexistent_attribute_with_default_value_can_be_saved (unexpected)
		--- FAIL: RelationTest#test_finding_with_reorder_by_aliased_attributes (unexpected)
		--- FAIL: RelationTest#test_finding_with_complex_order (unexpected)
		--- FAIL: HasManyAssociationsTest#test_has_many_build_with_options (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_changing_defaults (unexpected)
		--- FAIL: HasOneAssociationsTest#test_has_one_with_touch_option_on_touch (unexpected)
		--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_native_decimal_insert_manual_vs_automatic (unexpected)
		--- FAIL: AssociationsExtensionsTest#test_association_with_default_scope (unexpected)
		--- FAIL: HasManyAssociationsTest#test_sti_subselect_count (unexpected)
		--- FAIL: RelationTest#test_find_all_using_where_with_relation_with_bound_values (unexpected)
		--- FAIL: ActiveRecord::CustomPropertiesTest#test_procs_for_default_values_are_evaluated_even_after_column_defaults_is_called (unexpected)
		--- FAIL: HasManyAssociationsTest#test_dependence_for_associations_with_hash_condition (unexpected)
		--- FAIL: HasManyAssociationsTest#test_select_with_block (unexpected)
		--- FAIL: StrictLoadingTest#test_does_not_raise_on_eager_loading_a_strict_loading_habtm_relation (unexpected)
		--- FAIL: HasManyAssociationsTest#test_adding_using_create (unexpected)
		--- FAIL: RelationTest#test_order_triggers_eager_loading_when_ordering_using_symbols (unexpected)
		--- FAIL: HasManyAssociationsTest#test_delete_all_with_option_delete_all (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6775433?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6775433?buildTab=artifacts#/activerecord) on master @ [0eaeeb773474716753781289788fdd087fb9b166](https://github.com/cockroachdb/cockroach/commits/0eaeeb773474716753781289788fdd087fb9b166):


```
test artifacts and logs in: /artifacts/activerecord/run_1
	orm_helpers.go:191,orm_helpers.go:117,activerecord.go:248,test_runner.go:928: 
		Tests run on Cockroach v22.2.0-alpha.3-887-g0eaeeb7734
		Tests run against activerecord 7.0.3
		7347 Total Tests Run
		7334 tests passed
		13 tests failed
		26 tests skipped
		5 tests ignored
		0 tests passed unexpectedly
		13 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
		--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
		--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
		--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
		--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
		--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
		--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
		--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
		For a full summary look at the activerecord artifacts 
		
		An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6784051?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6784051?buildTab=artifacts#/activerecord) on master @ [d090ac9e42263b3fe8ec94d156bed056d06bedac](https://github.com/cockroachdb/cockroach/commits/d090ac9e42263b3fe8ec94d156bed056d06bedac):


```
--- FAIL: AssociationsJoinModelTest#test_eager_load_has_many_through_has_many (unexpected)
--- FAIL: AssociationsJoinModelTest#test_calculations_on_has_many_through_should_not_disambiguate_fields_unless_necessary (unexpected)
--- FAIL: AssociationsJoinModelTest#test_eager_load_has_many_through_has_many_with_conditions (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_set_record_to_nil_should_delete_association (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_set_record_after_delete_association (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_preloaded_records_are_not_duplicated (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PrimaryKeyAnyTypeTest#test_any_type_primary_key (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_nested_has_many_through_with_conditions_on_through_associations_preload (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_polymorphic_has_many_through_when_through_association_has_already_loaded (unexpected)
--- FAIL: HasManyThroughDisableJoinsAssociationsTest#test_count_on_disable_joins_through_a_through (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_up (unexpected)
--- FAIL: HasManyThroughDisableJoinsAssociationsTest#test_first_and_scope_applied_in_double_join (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_has_many_through_has_one_through_with_has_one_source_reflection_preload (unexpected)
--- FAIL: HasManyThroughDisableJoinsAssociationsTest#test_limit_and_scope_in_double_join_applies_limit_in_memory (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_has_many_through_has_many_through_with_belongs_to_source_reflection_preload (unexpected)
--- FAIL: FixturesTest#test_yml_file_in_subdirectory (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_preloading_across_has_one_constrains_loaded_records (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_polymorphic (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_cascaded_interdependent_one_level_and_two_levels (unexpected)
--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_going_through_join_model_with_inheritance (unexpected)
--- FAIL: HasManyThroughDisableJoinsAssociationsTest#test_empty_on_disable_joins_through (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_belongs_to_sti (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_distinct_has_many_through_a_has_many_through_association_on_through_reflection (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_has_one_through_belongs_to_setting_belongs_to_foreign_key_after_nil_target_loaded (unexpected)
--- FAIL: AssociationsJoinModelTest#test_associating_unsaved_records_with_has_many_through (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_has_many_find_first (unexpected)
--- FAIL: HasManyThroughDisableJoinsAssociationsTest#test_to_a_on_disable_joins_through (unexpected)
--- FAIL: AutomaticInverseFindingTests#test_has_one_and_belongs_to_with_non_default_foreign_key_should_not_find_inverse_automatically (unexpected)
--- FAIL: FixturesTest#test_empty_yaml_fixture_with_a_comment_in_it (unexpected)
--- FAIL: AssociationsJoinModelTest#test_delete_associate_when_deleting_from_has_many_through (unexpected)
--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyTest#test_references_foreign_key_with_suffix (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_creating_association_sets_both_parent_ids_for_new (unexpected)
--- FAIL: HasManyThroughDisableJoinsAssociationsTest#test_to_a_on_disable_joins_with_multiple_scopes (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_preloading_has_one_through_on_belongs_to (unexpected)
--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyInCreateTest#test_to_table_option_can_be_passed (unexpected)
--- FAIL: HasManyThroughDisableJoinsAssociationsTest#test_pluck_on_disable_joins_through_using_custom_foreign_key (unexpected)
--- FAIL: ActiveRecord::Migration::ReferencesForeignKeyInCreateTest#test_options_hash_can_be_passed (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_has_many_through_has_many_with_has_and_belongs_to_many_source_reflection (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_with_custom_primary_key_on_belongs_to_source (unexpected)
--- FAIL: FixturesTest#test_no_args_record_returns_all_without_array (unexpected)
--- FAIL: FixturesTest#test_serialized_fixtures (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: AssociationsJoinModelTest#test_belongs_to_polymorphic_with_counter_cache (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_reassigning_has_one_through (unexpected)
--- FAIL: ActiveRecord::InvertibleMigrationTest#test_migrate_revert_change_table (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_through_belongs_to_after_destroy (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6803698?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6803698?buildTab=artifacts#/activerecord) on master @ [1f7d3baebdee811380b4f3e8a0d634701041c5d0](https://github.com/cockroachdb/cockroach/commits/1f7d3baebdee811380b4f3e8a0d634701041c5d0):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-960-g1f7d3baebd
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0 release-blocker]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6824043?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6824043?buildTab=artifacts#/activerecord) on master @ [ded91681f2be8816238a05850170bde7f61b209f](https://github.com/cockroachdb/cockroach/commits/ded91681f2be8816238a05850170bde7f61b209f):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1010-gded91681f2
Tests run against activerecord 7.0.3
7347 Total Tests Run
7333 tests passed
14 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
14 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: CockroachDB::FixturesTest#test_bulk_insert (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6837129?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6837129?buildTab=artifacts#/activerecord) on master @ [e06d2286b011096526eda7f2d7f7bb7acea0ae84](https://github.com/cockroachdb/cockroach/commits/e06d2286b011096526eda7f2d7f7bb7acea0ae84):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1023-ge06d2286b0
Tests run against activerecord 7.0.3
7347 Total Tests Run
7323 tests passed
24 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
24 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_reference_column_of_child_table (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_column_of_child_table (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_reference_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_change_column_of_parent_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_change_column_of_parent_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_reference_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_reference_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6848253?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6848253?buildTab=artifacts#/activerecord) on master @ [e06d2286b011096526eda7f2d7f7bb7acea0ae84](https://github.com/cockroachdb/cockroach/commits/e06d2286b011096526eda7f2d7f7bb7acea0ae84):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1023-ge06d2286b0
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_simple_interval (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6859829?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6859829?buildTab=artifacts#/activerecord) on master @ [6e06a05eba118f512087da9aec3147b3446c2faf](https://github.com/cockroachdb/cockroach/commits/6e06a05eba118f512087da9aec3147b3446c2faf):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1027-g6e06a05eba
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6879260?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6879260?buildTab=artifacts#/activerecord) on master @ [8a8855eaf1a34223523d7631600481f1c2577a96](https://github.com/cockroachdb/cockroach/commits/8a8855eaf1a34223523d7631600481f1c2577a96):


```
--- FAIL: HasManyAssociationsTest#test_does_not_duplicate_associations_when_used_with_natural_primary_keys (unexpected)
--- FAIL: OptimisticLockingTest#test_non_integer_lock_destroy (unexpected)
--- FAIL: HasManyAssociationsTest#test_has_many_through_respects_hash_conditions (unexpected)
--- FAIL: HasManyAssociationsTest#test_included_in_collection (unexpected)
--- FAIL: HasManyAssociationsTest#test_has_many_association_with_same_foreign_key_name (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_default_scope (unexpected)
--- FAIL: HasManyAssociationsTest#test_taking_with_a_number (unexpected)
--- FAIL: HasManyAssociationsTest#test_in_memory_replacement_executes_no_queries (unexpected)
--- FAIL: EagerAssociationTest#test_works_in_combination_with_order(:symbol)_and_reorder(:symbol) (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
--- FAIL: HasManyAssociationsTest#test_deleting_updates_counter_cache_with_dependent_delete_all (unexpected)
--- FAIL: InheritanceTest#test_a_bad_type_column (unexpected)
--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_limit (unexpected)
--- FAIL: HasManyAssociationsTest#test_transactions_when_adding_to_new_record (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_column_name_existing (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_with_custom_primary_key (unexpected)
--- FAIL: HasManyAssociationsTest#test_find_first_after_reload (unexpected)
--- FAIL: HasManyAssociationsTest#test_reloading_unloaded_associations_with_query_cache (unexpected)
--- FAIL: DDLTest#test_add_geometry_column_null_false (unexpected)
--- FAIL: HasManyAssociationsTest#test_adding (unexpected)
--- FAIL: HasManyAssociationsTest#test_dynamic_find_should_respect_association_order_for_through (unexpected)
--- FAIL: HasManyAssociationsTest#test_find_ids_and_inverse_of (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_with_touch_on_multiple_records (unexpected)
--- FAIL: InheritanceTest#test_inheritance_with_default_scope (unexpected)
--- FAIL: HasManyAssociationsTest#test_included_in_collection_for_new_records (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_symbol_column (unexpected)
--- FAIL: EagerAssociationTest#test_eager_load_belongs_to_quotes_table_and_column_names (unexpected)
--- FAIL: EagerAssociationTest#test_preloading_the_same_association_twice_works (unexpected)
--- FAIL: EagerAssociationTest#test_has_many_through_with_order (unexpected)
--- FAIL: RelationScopingTest#test_scoped_find_combines_and_sanitizes_conditions (unexpected)
--- FAIL: HasManyAssociationsTest#test_delete_all,_when_not_loaded,_doesn't_load_the_records (unexpected)
--- FAIL: AssociationProxyTest#test_save_on_parent_does_not_load_target (unexpected)
--- FAIL: BelongsToAssociationsTest#test_tracking_change_from_nil_to_persisted_record (unexpected)
--- FAIL: EagerAssociationTest#test_preload_has_many_using_primary_key (unexpected)
--- FAIL: HasManyAssociationsTest#test_build_from_association_sets_inverse_instance (unexpected)
--- FAIL: OptimisticLockingTest#test_lock_with_custom_column_without_default_should_work_with_null_in_the_database (unexpected)
--- FAIL: HasManyAssociationsTest#test_replace_with_same_content (unexpected)
--- FAIL: EagerAssociationTest#test_order_on_join_table_with_include_and_limit (unexpected)
--- FAIL: UpdateAllTest#test_update_all_with_group_by (unexpected)
--- FAIL: BelongsToAssociationsTest#test_belongs_to_proxy_should_not_respond_to_private_methods (unexpected)
--- FAIL: HasManyAssociationsTest#test_delete_all_with_option_delete_all (unexpected)
--- FAIL: AssociationProxyTest#test_pick_uses_loaded_target (unexpected)
--- FAIL: HasManyAssociationsTest#test_attributes_are_set_when_initialized_from_has_many_null_relationship (unexpected)
--- FAIL: HasManyAssociationsTest#test_passes_custom_context_validation_to_validate_children (unexpected)
--- FAIL: BelongsToAssociationsTest#test_where_on_polymorphic_association_with_nil (unexpected)
--- FAIL: HasManyAssociationsTest#test_with_polymorphic_has_many_with_custom_columns_name (unexpected)
--- FAIL: HasManyAssociationsTest#test_abstract_class_with_polymorphic_has_many (unexpected)
--- FAIL: RelationScopingTest#test_reverse_order_with_arel_node (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6897503?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6897503?buildTab=artifacts#/activerecord) on master @ [d1962deb662c115ceec158a0c09d3f797eca3e71](https://github.com/cockroachdb/cockroach/commits/d1962deb662c115ceec158a0c09d3f797eca3e71):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1102-gd1962deb66
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6918230?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6918230?buildTab=artifacts#/activerecord) on master @ [4c2c7da2abef4875c44ade99fd4781c85be69637](https://github.com/cockroachdb/cockroach/commits/4c2c7da2abef4875c44ade99fd4781c85be69637):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1134-g4c2c7da2ab
Tests run against activerecord 7.0.3
7347 Total Tests Run
7335 tests passed
12 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
12 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6937671?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6937671?buildTab=artifacts#/activerecord) on master @ [d6c28f56d38c747dbd6fc64359e6255c873083c5](https://github.com/cockroachdb/cockroach/commits/d6c28f56d38c747dbd6fc64359e6255c873083c5):


```
--- FAIL: PrimaryKeysTest#test_update_columns_with_non_primary_key_id_column (unexpected)
--- FAIL: InheritanceTest#test_new_with_invalid_type (unexpected)
--- FAIL: ManyToManyFixturesWithClassDefined#test_this_should_run_cleanly (unexpected)
--- FAIL: BelongsToAssociationsTest#test_polymorphic_counter_cache (unexpected)
--- FAIL: HasManyAssociationsTest#test_raises_RecordNotDestroyed_when_replaced_child_can't_be_destroyed (unexpected)
--- FAIL: RelationScopingTest#test_reverse_order_with_arel_node (unexpected)
--- FAIL: InheritanceTest#test_alt_complex_inheritance (unexpected)
--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_validation_does_not_validate_non_dirty_association_target (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_extra_options_are_forwarded_to_the_type_caster_constructor (unexpected)
--- FAIL: HasManyAssociationsTest#test_dependent_association_respects_optional_sanitized_conditions_on_delete (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_children_can_override_parents (unexpected)
--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_add_column_without_limit (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_procs_for_default_values_are_evaluated_even_after_column_defaults_is_called (unexpected)
--- FAIL: RelationScopingTest#test_scoping_applies_to_delete_with_all_queries (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_unknown_timestamp_column (unexpected)
--- FAIL: HasManyAssociationsTest#test_create_many (unexpected)
--- FAIL: HasManyAssociationsTest#test_update_all_respects_association_scope (unexpected)
--- FAIL: InheritanceTest#test_where_create_bang_with_unrelated_type (unexpected)
--- FAIL: BooleanTest#test_boolean_cast_from_string (unexpected)
--- FAIL: BelongsToAssociationsTest#test_build_with_block (unexpected)
--- FAIL: FixturesTest#test_empty_yaml_fixture (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_association_with_extend_option (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_size_on_an_association_that_has_been_loaded_does_not_perform_query (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_children_inherit_custom_properties (unexpected)
--- FAIL: ActiveRecord::CustomPropertiesTest#test_time_zone_aware_attribute (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_with_custom_timestamp_column (unexpected)
--- FAIL: ActiveRecord::ConnectionAdapters::LegacyConnectionHandlersShardingDbTest#test_sharding_separation (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_one_should_return_false_if_zero (unexpected)
--- FAIL: HasManyAssociationsTest#test_counter_cache_updates_in_memory_after_create_with_array (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_new_aliased_to_build (unexpected)
--- FAIL: InheritanceTest#test_compute_type_success (unexpected)
--- FAIL: RelationScopingTest#test_nested_scoping_applies_with_all_queries_set (unexpected)
--- FAIL: BelongsToAssociationsTest#test_with_select (unexpected)
--- FAIL: BooleanTest#test_find_by_boolean_string (unexpected)
--- FAIL: HasManyAssociationsTest#test_destroy_on_association_clears_scope (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_symbol_join_table (unexpected)
--- FAIL: BelongsToAssociationsTest#test_raises_type_mismatch_with_namespaced_class (unexpected)
--- FAIL: PostGISTest#test_save_and_load_point_from_wkt (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_many_should_return_false_if_none_or_one (unexpected)
--- FAIL: InheritanceTest#test_alt_destroy_all_within_inheritance (unexpected)
--- FAIL: ModulesTest#test_associations_spanning_cross_modules (unexpected)
--- FAIL: InheritanceTest#test_compute_type_no_method_error (unexpected)
--- FAIL: HasManyAssociationsTest#test_has_many_with_out_of_range_value (unexpected)
--- FAIL: TestDefaultAutosaveAssociationOnABelongsToAssociation#test_store_association_in_two_relations_with_one_save_in_existing_object_with_values (unexpected)
--- FAIL: ActiveRecordSchemaTest#test_timestamps_without_null_set_null_to_false_on_add_timestamps (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_column (unexpected)
--- FAIL: InheritanceTest#test_inheritance_new_with_subclass (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_empty_with_counter_cache (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6949067?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6949067?buildTab=artifacts#/activerecord) on master @ [d6c28f56d38c747dbd6fc64359e6255c873083c5](https://github.com/cockroachdb/cockroach/commits/d6c28f56d38c747dbd6fc64359e6255c873083c5):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1172-gd6c28f56d3
Tests run against activerecord 7.0.3
7347 Total Tests Run
7335 tests passed
12 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
12 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6953743?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6953743?buildTab=artifacts#/activerecord) on master @ [7be0b20edbc336200c1510a9c6f1d76ae2f92c3a](https://github.com/cockroachdb/cockroach/commits/7be0b20edbc336200c1510a9c6f1d76ae2f92c3a):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1196-g7be0b20edb
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6964755?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6964755?buildTab=artifacts#/activerecord) on master @ [7be0b20edbc336200c1510a9c6f1d76ae2f92c3a](https://github.com/cockroachdb/cockroach/commits/7be0b20edbc336200c1510a9c6f1d76ae2f92c3a):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1196-g7be0b20edb
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::Encryption::StoragePerformanceTest#test_storage_overload_without_storing_keys_is_acceptable (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6975891?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6975891?buildTab=artifacts#/activerecord) on master @ [6b68e55309343a26300296bf42237a70d47297b2](https://github.com/cockroachdb/cockroach/commits/6b68e55309343a26300296bf42237a70d47297b2):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1200-g6b68e55309
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6994309?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/6994309?buildTab=artifacts#/activerecord) on master @ [762c1b86fe1c3a70338cee4a91c9e2e4c5e0fcfe](https://github.com/cockroachdb/cockroach/commits/762c1b86fe1c3a70338cee4a91c9e2e4c5e0fcfe):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1244-g762c1b86fe
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7014808?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7014808?buildTab=artifacts#/activerecord) on master @ [bb4f8cd5396e73c31d370c0d0ac1881263c5156f](https://github.com/cockroachdb/cockroach/commits/bb4f8cd5396e73c31d370c0d0ac1881263c5156f):


```
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_suffix (unexpected)
--- FAIL: HasOneAssociationsTest#test_creation_failure_due_to_new_record_should_raise_error (unexpected)
--- FAIL: HasOneAssociationsTest#test_natural_assignment (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_name (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_cascade_foreign_key (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_reference_column_of_child_table (unexpected)
--- FAIL: HasOneAssociationsTest#test_natural_assignment_to_already_associated_record (unexpected)
--- FAIL: HasOneAssociationsTest#test_build_with_block (unexpected)
--- FAIL: HasOneAssociationsTest#test_has_one_with_touch_option_on_create (unexpected)
--- FAIL: HasOneAssociationsTest#test_attributes_are_being_set_when_initialized_from_has_one_association_with_where_clause (unexpected)
--- FAIL: HasOneAssociationsTest#test_dependency_should_halt_parent_destruction (unexpected)
--- FAIL: HasOneAssociationsTest#test_nullification_on_association_change (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_inferes_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_name (unexpected)
--- FAIL: HasOneAssociationsTest#test_create_association_replaces_existing_without_dependent_option (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: HasOneAssociationsTest#test_replacement_failure_due_to_existing_record_should_raise_error (unexpected)
--- FAIL: HasOneAssociationsTest#test_association_keys_bypass_attribute_protection (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_constraint_by_name (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_change_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ReferencesForeignKeyTest#test_foreign_key_column_can_be_removed (unexpected)
--- FAIL: HasOneAssociationsTest#test_dangerous_association_name_raises_ArgumentError (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_prefix (unexpected)
--- FAIL: HasOneAssociationsTest#test_has_one_with_touch_option_on_nonpersisted_built_associations_doesnt_update_parent (unexpected)
--- FAIL: HasOneAssociationsTest#test_clearing_an_association_clears_the_associations_inverse (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_nullify_foreign_key (unexpected)
--- FAIL: HasOneAssociationsTest#test_has_one_assignment_triggers_save_on_change_on_replacing_object (unexpected)
--- FAIL: HasOneAssociationsTest#test_association_enum_works_properly (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_with_if_exists_set (unexpected)
--- FAIL: HasOneAssociationsTest#test_has_one_proxy_should_respond_to_private_methods_via_send (unexpected)
--- FAIL: HasOneAssociationsTest#test_has_one_loading_for_new_record (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: HasOneAssociationsTest#test_build_respects_hash_condition (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_name (unexpected)
--- FAIL: HasOneAssociationsTest#test_dependence_with_missing_association_and_nullify (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ReferencesForeignKeyTest#test_multiple_foreign_keys_can_be_removed_to_the_selected_one (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_restrict_foreign_key (unexpected)
--- FAIL: HasOneAssociationsTest#test_create (unexpected)
--- FAIL: HasOneAssociationsTest#test_finding_using_primary_key (unexpected)
--- FAIL: HasOneAssociationsTest#test_has_one_autosave_with_primary_key_manually_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: HasOneAssociationsTest#test_create_with_inexistent_foreign_key_failing (unexpected)
--- FAIL: CommentTest#test_comment_on_primary_key (unexpected)
--- FAIL: HasOneAssociationsTest#test_cant_save_readonly_association (unexpected)
--- FAIL: HasOneAssociationsTest#test_assignment_before_child_saved (unexpected)
--- FAIL: HasOneAssociationsTest#test_restrict_with_exception (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7033383?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7033383?buildTab=artifacts#/activerecord) on master @ [16b020292dbcdf9699da531764929ebdc8e48c43](https://github.com/cockroachdb/cockroach/commits/16b020292dbcdf9699da531764929ebdc8e48c43):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1326-g16b020292d
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7053206?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7053206?buildTab=artifacts#/activerecord) on master @ [0a11b74e2343e5201900fd2643724774fcf3fcd1](https://github.com/cockroachdb/cockroach/commits/0a11b74e2343e5201900fd2643724774fcf3fcd1):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1366-g0a11b74e23
Tests run against activerecord 7.0.3
7347 Total Tests Run
7335 tests passed
12 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
12 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7070913?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7070913?buildTab=artifacts#/activerecord) on master @ [a3b578466bcdcc5d2bbea628d33c337d87b9f4b8](https://github.com/cockroachdb/cockroach/commits/a3b578466bcdcc5d2bbea628d33c337d87b9f4b8):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1397-ga3b578466b
Tests run against activerecord 7.0.3
7347 Total Tests Run
7335 tests passed
12 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
12 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7082415?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7082415?buildTab=artifacts#/activerecord) on master @ [cdf4b18048a9e29019125c64bea75c8d50fa3774](https://github.com/cockroachdb/cockroach/commits/cdf4b18048a9e29019125c64bea75c8d50fa3774):


```
--- FAIL: OptimisticLockingTest#test_yaml_dumping_with_lock_column (unexpected)
--- FAIL: TestNestedAttributesLimitNumeric#test_limit_with_number_exact_records (unexpected)
--- FAIL: PersistenceTest#test_delete (unexpected)
--- FAIL: QueryCacheTest#test_writes_should_always_clear_cache (unexpected)
--- FAIL: PersistenceTest#test_create_many_through_factory_with_block (unexpected)
--- FAIL: PersistenceTest#test_delete_record_with_associations (unexpected)
--- FAIL: OptimisticLockingWithSchemaChangeTest#test_destroy_existing_object_with_locking_column_value_null_in_the_database (unexpected)
--- FAIL: ModulesTest#test_eager_loading_in_modules (unexpected)
--- FAIL: PersistenceTest#test_becomes (unexpected)
--- FAIL: TestAutosaveAssociationValidationsOnAHasManyAssociation#test_validations_still_fire_on_unchanged_association_with_custom_validation_context (unexpected)
--- FAIL: BasicsTest#test_column_name_properly_quoted (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_caching_of_columns (unexpected)
--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_skip_validation_on_has_many_if_destroyed (unexpected)
--- FAIL: RelationTest#test_to_xml (unexpected)
--- FAIL: NestedAttributesWithCallbacksTest#test_Assignment_updates_records_in_target_when_not_loaded (unexpected)
--- FAIL: NestedAttributesWithCallbacksTest#test_Assignment_updates_records_in_target_when_not_loaded_and_callback_loads_target (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: TransactionalFixturesOnCustomConnectionTest#test_leaky_destroy (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_build_by_new_record (unexpected)
--- FAIL: TestAutosaveAssociationValidationsOnAHABTMAssociation#test_should_not_automatically_validate_associations_without_:validate_=>_true (unexpected)
--- FAIL: PersistenceTest#test_update_many_with_active_record_base_object (unexpected)
--- FAIL: BasicsTest#test_quoted_table_name_after_set_table_name (unexpected)
--- FAIL: BasicsTest#test_default_in_utc (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_deleting_all (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_habtm_respects_select (unexpected)
--- FAIL: QueryCacheExpiryTest#test_upsert_all (unexpected)
--- FAIL: ActiveRecord::CollectionCacheKeyTest#test_cache_key_for_relation_with_includes (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_singular_ids_are_reloaded_after_collection_concat (unexpected)
--- FAIL: FixturesTest#test_complete_instantiation (unexpected)
--- FAIL: QueryCacheTest#test_query_cache_dups_results_correctly (unexpected)
--- FAIL: BasicsTest#test_quoting_arrays (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_build (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_find_scoped_grouped_having (unexpected)
--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_run_remove_callback_procs_for_habtm (unexpected)
--- FAIL: ModulesTest#test_find_account_and_include_company (unexpected)
--- FAIL: PersistenceTest#test_update_many_with_duplicated_ids! (unexpected)
--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_a_child_marked_for_destruction_should_not_be_destroyed_twice (unexpected)
--- FAIL: AssociationsJoinModelTest#test_polymorphic_has_many_going_through_join_model_with_include_on_source_reflection (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_preloading_across_has_one_constrains_loaded_records (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_going_through_join_model_with_custom_foreign_key (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_find_conditions (unexpected)
--- FAIL: QueryCacheTest#test_exists_queries_with_cache (unexpected)
--- FAIL: FixturesTest#test_insert_with_datetime (unexpected)
--- FAIL: BasicsTest#test_changing_a_connection_handler_in_a_main_thread_does_not_poison_the_other_threads (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_recursive_cascading_four_levels_has_many_through (unexpected)
--- FAIL: CascadedEagerLoadingTest#test_eager_association_loading_with_cascaded_two_levels_and_one_level (unexpected)
--- FAIL: TestDestroyAsPartOfAutosaveAssociation#test_should_rollback_destructions_if_an_exception_occurred_while_saving_a_parent (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7094346?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7094346?buildTab=artifacts#/activerecord) on master @ [11e299bb1bec2f5666658393f14c12ebe11cc4a4](https://github.com/cockroachdb/cockroach/commits/11e299bb1bec2f5666658393f14c12ebe11cc4a4):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1406-g11e299bb1b
Tests run against activerecord 7.0.3
7347 Total Tests Run
7333 tests passed
14 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
14 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_simple_interval (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7114342?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7114342?buildTab=artifacts#/activerecord) on master @ [c5decaeb69155399647909163451f3d8ca7858b6](https://github.com/cockroachdb/cockroach/commits/c5decaeb69155399647909163451f3d8ca7858b6):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(orm_helpers.go:191).summarizeFailed: 
Tests run on Cockroach v22.2.0-alpha.3-1444-gc5decaeb69
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7136814?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7136814?buildTab=artifacts#/activerecord) on master @ [1b1c8da55be48c174b7b370b305f42622546209f](https://github.com/cockroachdb/cockroach/commits/1b1c8da55be48c174b7b370b305f42622546209f):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1471-g1b1c8da55b
Tests run against activerecord 7.0.3
2309 Total Tests Run
2292 tests passed
17 tests failed
13 tests skipped
1 test ignored
0 tests passed unexpectedly
17 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: CockroachDB::FixturesTest#test_yaml_file_with_symbol_columns (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: CockroachDB::FixturesTest#test_auto_value_on_primary_key (unexpected)
--- FAIL: CockroachDB::FixturesTest#test_inserts_with_pre_and_suffix (unexpected)
--- FAIL: ActiveRecord::Encryption::UniquenessValidationsTest#test_uniqueness_validations_work_when_using_old_encryption_schemes (unexpected)
--- FAIL: CockroachDB::FixturesTest#test_bulk_insert (unexpected)
--- FAIL: CockroachDB::FixturesTest#test_create_fixtures (unexpected)
--- FAIL: CockroachDB::FixturesResetPkSequenceTest#test_resets_to_min_pk_with_specified_pk_and_sequence (unexpected)
--- FAIL: CockroachDB::FixturesResetPkSequenceTest#test_resets_to_min_pk_with_default_pk_and_sequence (unexpected)
--- FAIL: CockroachDB::FixturesResetPkSequenceTest#test_create_fixtures_resets_sequences_when_not_cached (unexpected)
--- FAIL: CockroachDB::FixturesTest#test_clean_fixtures (unexpected)
--- FAIL: CockroachDB::FixturesTest#test_multiple_clean_fixtures (unexpected)
--- FAIL: CockroachDB::MarshalSerializationTest#test_deserializing_rails_6_1_marshal_with_loaded_association_cache (unexpected)
--- FAIL: ActiveRecord::Encryption::UniquenessValidationsTest#test_uniqueness_validations_work (unexpected)
--- FAIL: CockroachDB::FixturesTest#test_create_symbol_fixtures (unexpected)
--- FAIL: CockroachDB::MarshalSerializationTest#test_deserializing_rails_6_1_marshal_basic (unexpected)
--- FAIL: CockroachDB::FixturesTest#test_bulk_insert_with_a_multi_statement_query_raises_an_exception_when_any_insert_fails (unexpected)
--- FAIL: ActiveRecord::Encryption::UniquenessValidationsTest#test_uniqueness_validations_work_when_mixing_encrypted_an_unencrypted_data (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7161019?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7161019?buildTab=artifacts#/activerecord) on master @ [100452314211aedd1e295bba6979df71d9052629](https://github.com/cockroachdb/cockroach/commits/100452314211aedd1e295bba6979df71d9052629):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1549-g1004523142
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7182921?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7182921?buildTab=artifacts#/activerecord) on master @ [d39558f51ded0411e90ef1d8aa05106930b36667](https://github.com/cockroachdb/cockroach/commits/d39558f51ded0411e90ef1d8aa05106930b36667):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1592-gd39558f51d
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7201916?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7201916?buildTab=artifacts#/activerecord) on master @ [0c1c3e7777b28a30ebe41428fb173f0156e8968c](https://github.com/cockroachdb/cockroach/commits/0c1c3e7777b28a30ebe41428fb173f0156e8968c):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1630-g0c1c3e7777
Tests run against activerecord 7.0.3
7347 Total Tests Run
7328 tests passed
19 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
19 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_reference_column_of_child_table (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_reference_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_change_column_of_parent_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_column_of_child_table (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7213327?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7213327?buildTab=artifacts#/activerecord) on master @ [0c1c3e7777b28a30ebe41428fb173f0156e8968c](https://github.com/cockroachdb/cockroach/commits/0c1c3e7777b28a30ebe41428fb173f0156e8968c):


```
--- FAIL: InheritanceTest#test_where_new_with_invalid_type (unexpected)
--- FAIL: ActiveRecord::Migration::ColumnAttributesTest#test_add_column_with_precision_and_scale (unexpected)
--- FAIL: FinderTest#test_first (unexpected)
--- FAIL: PersistenceTest#test_save_destroyed_object (unexpected)
--- FAIL: FinderTest#test_hash_condition_find_with_array (unexpected)
--- FAIL: BelongsToAssociationsTest#test_setting_foreign_key_after_nil_target_loaded (unexpected)
--- FAIL: FinderTest#test_condition_local_time_interpolation_with_default_timezone_utc (unexpected)
--- FAIL: BasicsTest#test_find_keeps_multiple_order_values (unexpected)
--- FAIL: IntegrationTest#test_to_param_returns_nil_if_not_persisted (unexpected)
--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to_and_order_string_with_quoted_table_name (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_has_many_through_has_and_belongs_to_many_with_has_many_source_reflection_preload_via_joins (unexpected)
--- FAIL: BasicsTest#test_default_values_are_deeply_dupped (unexpected)
--- FAIL: FinderTest#test_last_bang_missing (unexpected)
--- FAIL: BasicsTest#test_dont_clear_inheritance_column_when_setting_explicitly (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_through_belongs_to_after_destroy (unexpected)
--- FAIL: FinderTest#test_third_to_last_have_primary_key_order_by_default (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_has_many_through_join_model_with_include (unexpected)
--- FAIL: PersistenceTest#test_save_for_record_with_only_primary_key_that_is_provided (unexpected)
--- FAIL: ReflectionTest#test_type (unexpected)
--- FAIL: TouchTest#test_create_turned_off (unexpected)
--- FAIL: FinderTest#test_third_to_last (unexpected)
--- FAIL: HasManyThroughDisableJoinsAssociationsTest#test_pluck_on_disable_joins_through_a_through (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_nested_has_many_through_with_foreign_key_option_on_the_source_reflection_through_reflection (unexpected)
--- FAIL: ActiveRecord::Migration::ColumnsTest#test_change_column_nullability (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_uniqueness_with_scope_invalid_syntax (unexpected)
--- FAIL: AssociationProxyTest#test_append_behaves_like_push (unexpected)
--- FAIL: FinderTest#test_dynamic_finder_on_one_attribute_with_conditions_returns_same_results_after_caching (unexpected)
--- FAIL: PersistenceTest#test_destroy_many_with_invalid_id (unexpected)
--- FAIL: FinderTest#test_hash_condition_find_with_aggregate_having_three_mappings (unexpected)
--- FAIL: BasicsTest#test_ignored_columns_not_included_in_SELECT (unexpected)
--- FAIL: FinderTest#test_find_by_with_large_number (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: FinderTest#test_exists_with_distinct_and_offset_and_eagerload_and_order (unexpected)
--- FAIL: EagerAssociationTest#test_three_level_nested_preloading_does_not_raise_exception_when_association_does_not_exist (unexpected)
--- FAIL: InheritanceTest#test_alt_inheritance_find_all (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_nested_has_many_through_with_conditions_on_source_associations (unexpected)
--- FAIL: TransactionCallbacksTest#test_save_in_after_create_commit_wont_invoke_extra_after_create_commit (unexpected)
--- FAIL: FinderTest#test_hash_condition_find_nil_with_aggregate_having_one_mapping (unexpected)
--- FAIL: IntegrationTest#test_cache_key_is_stable_with_versioning_on (unexpected)
--- FAIL: InheritanceTest#test_eager_load_belongs_to_something_inherited (unexpected)
--- FAIL: PersistenceTest#test_destroyed_returns_boolean (unexpected)
--- FAIL: EagerAssociationTest#test_eager_with_floating_point_numbers (unexpected)
--- FAIL: RelationScopingTest#test_scoped_find (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: FinderTest#test_hash_condition_find_with_aggregate_having_one_mapping_and_key_value_being_attribute_value (unexpected)
--- FAIL: UpdateAllTest#test_update_all_cares_about_optimistic_locking (unexpected)
--- FAIL: BasicsTest#test_columns_should_obey_set_primary_key (unexpected)
--- FAIL: EagerAssociationTest#test_nested_loading_through_has_one_association_with_order_on_association (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7225361?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7225361?buildTab=artifacts#/activerecord) on master @ [2d926e68000df659f282d4e4477329867b9a3323](https://github.com/cockroachdb/cockroach/commits/2d926e68000df659f282d4e4477329867b9a3323):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1636-g2d926e6800
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7247923?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7247923?buildTab=artifacts#/activerecord) on master @ [18675b7268a89dc646f37b7208c2eaa319f7741c](https://github.com/cockroachdb/cockroach/commits/18675b7268a89dc646f37b7208c2eaa319f7741c):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1703-g18675b7268
Tests run against activerecord 7.0.3
7347 Total Tests Run
7335 tests passed
12 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
12 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7269456?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7269456?buildTab=artifacts#/activerecord) on master @ [bec24eb9cfe2cc48c45772f1e88d387fe37944c9](https://github.com/cockroachdb/cockroach/commits/bec24eb9cfe2cc48c45772f1e88d387fe37944c9):


```
--- FAIL: QueryCacheTest#test_query_cache_with_forked_processes (unexpected)
--- FAIL: ReflectionTest#test_join_table_with_common_prefix (unexpected)
--- FAIL: RelationTest#test_finding_with_complex_order (unexpected)
--- FAIL: CalculationsTest#test_count_with_eager_loading_and_custom_order_and_distinct (unexpected)
--- FAIL: RelationTest#test_find_by_id_with_list_of_ar (unexpected)
--- FAIL: FinderTest#test_find_some_message_with_custom_primary_key (unexpected)
--- FAIL: HasOneAssociationsTest#test_create_before_save (unexpected)
--- FAIL: FixturesTest#test_serialized_fixtures (unexpected)
--- FAIL: PersistenceTest#test_update_sti_subclass_type (unexpected)
--- FAIL: EagerAssociationTest#test_eager_loading_with_select_on_joined_table_preloads (unexpected)
--- FAIL: CalculationsTest#test_should_perform_joined_include_when_referencing_included_tables (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_has_many_through_with_left_joined_same_table_with_through_table (unexpected)
--- FAIL: EagerAssociationTest#test_order_on_join_table_with_include_and_limit (unexpected)
--- FAIL: FinderTest#test_find_by_array_of_one_id (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_should_raise_exception_for_destroying_mismatching_records (unexpected)
--- FAIL: RelationTest#test_grouping_by_column_with_reserved_name (unexpected)
--- FAIL: OptimisticLockingTest#test_non_integer_lock_destroy (unexpected)
--- FAIL: RelationTest#test_finding_with_group (unexpected)
--- FAIL: RelationTest#test_unscope_with_blank_value (unexpected)
--- FAIL: PersistenceTest#test_delete_isnt_affected_by_scoping (unexpected)
--- FAIL: CalculationsTest#test_pluck (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_get_ids_for_has_many_through_with_conditions_should_not_preload (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_value_is_properly_quoted (unexpected)
--- FAIL: RelationTest#test_order_triggers_eager_loading_when_ordering_using_hash_syntax (unexpected)
--- FAIL: BasicsTest#test_create_after_initialize_without_block (unexpected)
--- FAIL: CalculationsTest#test_should_average_field (unexpected)
--- FAIL: PersistenceTest#test_increment_with_no_arg (unexpected)
--- FAIL: PersistenceTest#test_delete_new_record (unexpected)
--- FAIL: PersistenceTest#test_update_all (unexpected)
--- FAIL: HasOneAssociationsTest#test_association_change_calls_destroy (unexpected)
--- FAIL: HasOneAssociationsTest#test_nullify_on_polymorphic_association (unexpected)
--- FAIL: PersistenceTest#test_save_for_record_with_only_primary_key (unexpected)
--- FAIL: CalculationsTest#test_pluck_not_auto_table_name_prefix_if_column_included (unexpected)
--- FAIL: FinderTest#test_find_by_title_and_id_with_hash (unexpected)
--- FAIL: InheritanceTest#test_compute_type_success (unexpected)
--- FAIL: FinderTest#test_exists_with_empty_hash_arg (unexpected)
--- FAIL: FinderTest#test_find_one_message_on_primary_key (unexpected)
--- FAIL: NestedThroughAssociationsTest#test_has_many_through_has_many_with_has_many_through_source_reflection_preload_via_joins (unexpected)
--- FAIL: FinderTest#test_hash_condition_find_malformed (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_no_pk_join_table_delete (unexpected)
--- FAIL: PrimaryKeysTest#test_update_counters_should_quote_pkey_and_quote_counter_columns (unexpected)
--- FAIL: RelationTest#test_typecasting_where_with_array (unexpected)
--- FAIL: RelationTest#test_where_with_take_memoization (unexpected)
--- FAIL: BasicsTest#test_previously_changed_dup (unexpected)
--- FAIL: RelationTest#test_raising_exception_on_invalid_hash_params (unexpected)
--- FAIL: QueryCacheTest#test_cache_does_not_wrap_results_in_arrays (unexpected)
--- FAIL: InsertAllTest#test_upsert_all_has_many_through (unexpected)
--- FAIL: HasOneAssociationsTest#test_dependence_with_missing_association (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7288517?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7288517?buildTab=artifacts#/activerecord) on master @ [9c9d55d707ad9a768027e9b7a3775c7c7cde8de7](https://github.com/cockroachdb/cockroach/commits/9c9d55d707ad9a768027e9b7a3775c7c7cde8de7):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v22.2.0-alpha.3-1768-g9c9d55d707
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7309179?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7309179?buildTab=artifacts#/activerecord) on master @ [01cfd27375aaf4633b6f6722924c8e547af4959d](https://github.com/cockroachdb/cockroach/commits/01cfd27375aaf4633b6f6722924c8e547af4959d):


```
--- FAIL: AssociationsJoinModelTest#test_calculations_on_has_many_through_should_not_disambiguate_fields_unless_necessary (unexpected)
--- FAIL: CalculationsTest#test_pluck_with_serialization (unexpected)
--- FAIL: FixturesTest#test_serialized_fixtures (unexpected)
--- FAIL: FinderTest#test_take_failing (unexpected)
--- FAIL: BasicsTest#test_primary_key_and_references_columns_should_be_identical_type (unexpected)
--- FAIL: InheritanceTest#test_becomes_sets_variables_before_initialization_callbacks (unexpected)
--- FAIL: AssociationsJoinModelTest#test_polymorphic_belongs_to (unexpected)
--- FAIL: EnumTest#test_overriding_enum_method_should_not_raise (unexpected)
--- FAIL: StoreTest#test_accessing_attributes_not_exposed_by_accessors (unexpected)
--- FAIL: BasicsTest#test_invalid_limit (unexpected)
--- FAIL: PersistenceTest#test_update_columns_should_raise_exception_if_new_record (unexpected)
--- FAIL: HasOneAssociationsTest#test_natural_assignment (unexpected)
--- FAIL: BasicsTest#test_benchmark_with_log_level (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::CheckConstraintTest#test_remove_check_constraint (unexpected)
--- FAIL: RelationScopingTest#test_scoped_default_scoped (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_reference_column_of_child_table (unexpected)
--- FAIL: CalculationsTest#test_limit_should_apply_before_count (unexpected)
--- FAIL: EagerAssociationTest#test_calculate_with_string_in_from_and_eager_loading (unexpected)
--- FAIL: FixturesTest#test_no_args_record_returns_all_without_array (unexpected)
--- FAIL: PersistenceTest#test_class_level_delete (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_attributes_are_being_set_when_initialized_from_hm_through_association_with_where_clause (unexpected)
--- FAIL: ReflectionTest#test_has_and_belongs_to_many_reflection (unexpected)
--- FAIL: ActiveRecord::LoadAsyncTest#test_pluck (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_belongs_to_with_dependent_destroy (unexpected)
--- FAIL: FinderTest#test_find_with_ids_with_id_out_of_range (unexpected)
--- FAIL: ActiveRecord::CacheKeyTest#test_cache_version_calls_updated_at_when_the_value_is_generated_at_create_time (unexpected)
--- FAIL: ActiveRecord::Relation::WhereClauseTest#test_except_removes_binary_predicates_referencing_a_given_column (unexpected)
--- FAIL: ConcurrentTransactionTest#test_transaction_state_is_cleared_when_record_is_persisted (unexpected)
--- FAIL: InheritanceTest#test_where_create_with_invalid_type (unexpected)
--- FAIL: ReflectionTest#test_non_existent_types_are_identity_types (unexpected)
--- FAIL: BasicsTest#test_find_by_slug_with_array (unexpected)
--- FAIL: TimestampTest#test_changing_parent_of_a_record_touches_both_new_and_old_polymorphic_parent_record_changes_within_same_class (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_delete_through_belongs_to_with_dependent_destroy (unexpected)
--- FAIL: HasOneAssociationsTest#test_association_keys_bypass_attribute_protection (unexpected)
--- FAIL: ValidationsTest#test_invalid_record_exception (unexpected)
--- FAIL: ViewWithPrimaryKeyTest#test_table_exists (unexpected)
--- FAIL: ActiveRecord::FixtureSet::FileTest#test_removes_fixture_config_row (unexpected)
--- FAIL: InversePolymorphicBelongsToTests#test_trying_to_set_polymorphic_inverses_that_dont_exist_at_all_should_raise_an_error (unexpected)
--- FAIL: HasOneAssociationsTest#test_has_one_proxy_should_respond_to_private_methods_via_send (unexpected)
--- FAIL: EnumTest#test_find_via_where_should_be_type_casted (unexpected)
--- FAIL: AssociationsJoinModelTest#test_delete_associate_when_deleting_from_has_many_through_with_nonstandard_id (unexpected)
--- FAIL: ActiveRecord::Relation::WhereClauseTest#test_an_empty_where_clause_is_the_identity_value_for_+ (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_collection_singular_ids_through_setter_raises_exception_when_invalid_ids_set (unexpected)
--- FAIL: ActiveRecord::ConnectionAdapters::PostgreSQLAdapter::QuotingTest#test_quote_range (unexpected)
--- FAIL: EagerAssociationTest#test_eager_association_loading_with_belongs_to (unexpected)
--- FAIL: AssociationsJoinModelTest#test_has_many_through_has_many_find_all_with_custom_class (unexpected)
--- FAIL: CalculationsTest#test_count_selected_arel_attribute (unexpected)
--- FAIL: PostgreSQLUtilsTest#test_extract_schema_qualified_name (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList22_2) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7330110?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7330110?buildTab=artifacts#/activerecord) on master @ [2c0129ce2507e708382cda830d1e47817a299c2e](https://github.com/cockroachdb/cockroach/commits/2c0129ce2507e708382cda830d1e47817a299c2e):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-35-g2c0129ce25
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7342303?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7342303?buildTab=artifacts#/activerecord) on master @ [c9094fa1a1ef8d44a2d5299d1d7380b34a06241b](https://github.com/cockroachdb/cockroach/commits/c9094fa1a1ef8d44a2d5299d1d7380b34a06241b):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-46-gc9094fa1a1
Tests run against activerecord 7.0.3
7347 Total Tests Run
7333 tests passed
14 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
14 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: CockroachDB::PostgresqlIntervalTest#test_simple_interval (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7354413?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7354413?buildTab=artifacts#/activerecord) on master @ [4c828ca5df8ea70b92ba58c6cb4868303cee1262](https://github.com/cockroachdb/cockroach/commits/4c828ca5df8ea70b92ba58c6cb4868303cee1262):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-48-g4c828ca5df
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7377852?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7377852?buildTab=artifacts#/activerecord) on master @ [ff8f7eda167887e1f11bbcd9029aa4cc2d28085e](https://github.com/cockroachdb/cockroach/commits/ff8f7eda167887e1f11bbcd9029aa4cc2d28085e):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-69-gff8f7eda16
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7402563?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7402563?buildTab=artifacts#/activerecord) on master @ [ee4fdb7fff683a7916ee585ac0c1e22d5d9a4412](https://github.com/cockroachdb/cockroach/commits/ee4fdb7fff683a7916ee585ac0c1e22d5d9a4412):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-144-gee4fdb7fff
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7423823?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7423823?buildTab=artifacts#/activerecord) on master @ [d6ae9653b67071cca09ed296711cecc821fc284e](https://github.com/cockroachdb/cockroach/commits/d6ae9653b67071cca09ed296711cecc821fc284e):


```
--- FAIL: FinderTest#test_find_on_hash_conditions_with_multiple_ranges (unexpected)
--- FAIL: FinderTest#test_find_by_title_and_id_with_hash (unexpected)
--- FAIL: FinderTest#test_exists_with_distinct_association_includes_limit_and_order (unexpected)
--- FAIL: FinderTest#test_hash_condition_find_with_aggregate_having_one_mapping_array (unexpected)
--- FAIL: FinderTest#test_exists_with_distinct_and_offset_and_select (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_1#test_change_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: FinderTest#test_find_with_string (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_destruction_does_not_error_without_primary_key (unexpected)
--- FAIL: StrictLoadingTest#test_does_not_raise_on_eager_loading_a_strict_loading_has_one_relation (unexpected)
--- FAIL: FinderTest#test_sole_failing_many (unexpected)
--- FAIL: FinderTest#test_implicit_order_set_to_primary_key (unexpected)
--- FAIL: FinderTest#test_include_on_unloaded_relation_with_match (unexpected)
--- FAIL: FinderTest#test_find_on_hash_conditions_with_explicit_table_name_and_aggregate (unexpected)
--- FAIL: PostgresqlBitStringTest#test_roundtrip (unexpected)
--- FAIL: FinderTest#test_exists_passing_active_record_object_is_not_permitted (unexpected)
--- FAIL: StrictLoadingTest#test_strict_loading_has_one_reload (unexpected)
--- FAIL: ActiveRecord::Migration::ColumnsTest#test_add_rename (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_redefine_habtm (unexpected)
--- FAIL: FinderTest#test_find_by!_with_non-hash_conditions_returns_the_first_matching_record (unexpected)
--- FAIL: CockroachDB::PostgresqlTimestampMigrationTest#test_adds_column_as_timestamp (unexpected)
--- FAIL: FinderTest#test_hash_condition_find_with_array (unexpected)
--- FAIL: FinderTest#test_member_on_unloaded_relation_without_match (unexpected)
--- FAIL: FinderTest#test_#skip_query_cache!_for_#exists?_with_a_limited_eager_load (unexpected)
--- FAIL: FinderTest#test_third_with_offset (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_associations_with_conditions (unexpected)
--- FAIL: ModulesTest#test_assign_ids (unexpected)
--- FAIL: FinderTest#test_find_on_hash_conditions_with_numeric_range_for_string (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_habtm_collection_size_from_params (unexpected)
--- FAIL: FinderTest#test_find_by_with_multi-arg_conditions_returns_the_first_matching_record (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_new_aliased_to_build (unexpected)
--- FAIL: FinderTest#test_find_by_array_of_one_id (unexpected)
--- FAIL: FinderTest#test_find_by_with_non-hash_conditions_returns_the_first_matching_record (unexpected)
--- FAIL: FinderTest#test_second (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_set (unexpected)
--- FAIL: FinderTest#test_exists_should_reference_correct_aliases_while_joining_tables_of_has_many_through_association (unexpected)
--- FAIL: StrictLoadingTest#test_strict_loading_violation_can_log_instead_of_raise (unexpected)
--- FAIL: FinderTest#test_first_bang_missing (unexpected)
--- FAIL: FinderTest#test_find_on_hash_conditions_with_array_of_integers_and_ranges (unexpected)
--- FAIL: CockroachDB::FixturesResetPkSequenceTest#test_create_fixtures_resets_sequences_when_not_cached (unexpected)
--- FAIL: FinderTest#test_model_class_responds_to_second_to_last_bang (unexpected)
--- FAIL: HasAndBelongsToManyAssociationsTest#test_has_and_belongs_to_many (unexpected)
--- FAIL: FinderTest#test_last_bang_missing (unexpected)
--- FAIL: PostgresqlBitStringTest#test_bit_string_varying_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping_on_delete_and_on_update_options (unexpected)
--- FAIL: StrictLoadingTest#test_raises_on_lazy_loading_a_has_one_relation_if_strict_loading_by_default (unexpected)
--- FAIL: FinderTest#test_find_by_doesn't_have_implicit_ordering (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7448348?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7448348?buildTab=artifacts#/activerecord) on master @ [d136c1fa50fa507c2232d7b215d98b88a83649c5](https://github.com/cockroachdb/cockroach/commits/d136c1fa50fa507c2232d7b215d98b88a83649c5):


```
--- FAIL: PrimaryKeysTest#test_to_key_with_customized_primary_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_column (unexpected)
--- FAIL: DDLTest#test_create_point_geometry (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_prefix (unexpected)
--- FAIL: PrimaryKeysTest#test_customized_primary_key_can_be_get_before_saving (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: DDLTest#test_create_simple_geometry (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_on_delete_restrict_foreign_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_constraint_is_not_cached_incorrectly (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: BidirectionalDestroyDependenciesTest#test_bidirectional_dependence_when_destroying_item_with_has_one_association_fails_first_time (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ReferencesForeignKeyTest#test_removing_column_removes_foreign_key (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_2#test_change_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: DDLTest#test_add_geography_column (unexpected)
--- FAIL: DDLTest#test_reload_dumped_schema (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: DDLTest#test_add_geometry_column (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_0#test_polymorphic_add_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_with_if_exists_not_set (unexpected)
--- FAIL: DDLTest#test_create_point_geometry_using_shortcut (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_foreign_key_exists_by_name (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_inferes_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_validate_foreign_key_by_name (unexpected)
--- FAIL: AttributeMethodsTest#test_id_before_type_cast_with_a_custom_primary_key (unexpected)
--- FAIL: BooleanTest#test_boolean_cast_from_string (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_2#test_create_join_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: AttributeMethodsTest#test_respond_to?_with_a_custom_primary_key (unexpected)
--- FAIL: DDLTest#test_create_simple_geography_using_shortcut (unexpected)
--- FAIL: DDLTest#test_non_spatial_column_limits (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_with_if_exists_set (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V5_0#test_create_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_if_not_exists_not_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_name (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PrimaryKeysTest#test_read_attribute_with_custom_primary_key (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ReferencesForeignKeyTest#test_multiple_foreign_keys_can_be_added_to_the_same_table (unexpected)
--- FAIL: DDLTest#test_drop_geometry_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: DDLTest#test_column_types (unexpected)
--- FAIL: DDLTest#test_create_geometry_using_shortcut_with_srid (unexpected)
--- FAIL: BidirectionalDestroyDependenciesTest#test_bidirectional_dependence_when_destroying_item_with_has_one_association (unexpected)
--- FAIL: CockroachDB::MigrationTest#test_remove_column_with_if_not_exists_not_set (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7466963?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7466963?buildTab=artifacts#/activerecord) on master @ [fb4014a31b9b8d8235dc48de52196e64b185f490](https://github.com/cockroachdb/cockroach/commits/fb4014a31b9b8d8235dc48de52196e64b185f490):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-258-gfb4014a31b
Tests run against activerecord 7.0.3
7347 Total Tests Run
7322 tests passed
25 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
25 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_reference_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_rename_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_reference_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_change_column_of_parent_table (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_reference_column_of_child_table (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_rename_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_rename_reference_column_of_child_table (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnTest#test_change_column_of_parent_table (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithSuffixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7478515?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7478515?buildTab=artifacts#/activerecord) on master @ [af18b5d2f8ec09d6f3c3092be9ef5fe3f460724d](https://github.com/cockroachdb/cockroach/commits/af18b5d2f8ec09d6f3c3092be9ef5fe3f460724d):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-265-gaf18b5d2f8
Tests run against activerecord 7.0.3
7347 Total Tests Run
7335 tests passed
12 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
12 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7490747?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7490747?buildTab=artifacts#/activerecord) on master @ [7f77f57a768a1bc488629234126e9e456bb6ddbb](https://github.com/cockroachdb/cockroach/commits/7f77f57a768a1bc488629234126e9e456bb6ddbb):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-269-g7f77f57a76
Tests run against activerecord 7.0.3
7347 Total Tests Run
7333 tests passed
14 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
14 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::Encryption::EncryptionPerformanceTest#test_performance_when_saving_records (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7512756?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7512756?buildTab=artifacts#/activerecord) on master @ [47c7b3a1bc047fc3e481cf12166885b39519c022](https://github.com/cockroachdb/cockroach/commits/47c7b3a1bc047fc3e481cf12166885b39519c022):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-310-g47c7b3a1bc
Tests run against activerecord 7.0.3
7347 Total Tests Run
7333 tests passed
14 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
14 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: BasicsTest#test_default_values_are_deeply_dupped (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7534902?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7534902?buildTab=artifacts#/activerecord) on master @ [d98d195dfa7a3480ce3e07657aa870692fe71cea](https://github.com/cockroachdb/cockroach/commits/d98d195dfa7a3480ce3e07657aa870692fe71cea):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-356-gd98d195dfa
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7555547?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7555547?buildTab=artifacts#/activerecord) on master @ [ea7c52e5a930be203318db6e338edd5dde7537bd](https://github.com/cockroachdb/cockroach/commits/ea7c52e5a930be203318db6e338edd5dde7537bd):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-425-gea7c52e5a9
Tests run against activerecord 7.0.3
7347 Total Tests Run
7332 tests passed
15 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
15 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: CockroachDB::FixturesTest#test_bulk_insert (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: BasicsTest#test_default_values_are_deeply_dupped (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7574283?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7574283?buildTab=artifacts#/activerecord) on master @ [a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412](https://github.com/cockroachdb/cockroach/commits/a035d2e6e7ea57b30115ecc8ee1a5d553a9e3412):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-474-ga035d2e6e7
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7593502?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7593502?buildTab=artifacts#/activerecord) on master @ [8357abb668a5adaff781343b394b162fb1b66c6e](https://github.com/cockroachdb/cockroach/commits/8357abb668a5adaff781343b394b162fb1b66c6e):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-506-g8357abb668
Tests run against activerecord 7.0.3
7347 Total Tests Run
7326 tests passed
21 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
21 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_not_save_and_return_false_if_a_callback_cancelled_saving_in_either_create_or_update (unexpected)
--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_work_with_update_as_well (unexpected)
--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_not_update_children_when_parent_creation_with_no_reason (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_not_load_association_when_updating_existing_records (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_take_an_array_and_assign_the_attributes_to_the_associated_models (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_also_work_with_a_HashWithIndifferentAccess (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: TestAutosaveAssociationOnAHasAndBelongsToManyAssociationWithAcceptsNestedAttributes#test_should_update_children_when_autosave_is_true_and_parent_is_new_but_child_is_not (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: TestNestedAttributesOnAHasAndBelongsToManyAssociation#test_should_take_a_hash_with_string_keys_and_assign_the_attributes_to_the_associated_models (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7605127?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7605127?buildTab=artifacts#/activerecord) on master @ [cfb5ae9a96e1770daa4aef1615a46e212b561a84](https://github.com/cockroachdb/cockroach/commits/cfb5ae9a96e1770daa4aef1615a46e212b561a84):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-516-gcfb5ae9a96
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7616997?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7616997?buildTab=artifacts#/activerecord) on master @ [f694132d6ed73c707576a28f6e784fda8ea13c63](https://github.com/cockroachdb/cockroach/commits/f694132d6ed73c707576a28f6e784fda8ea13c63):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-522-gf694132d6e
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7636747?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7636747?buildTab=artifacts#/activerecord) on master @ [dca415eddac0d659ae6d76b4e3dfdf4076adbd34](https://github.com/cockroachdb/cockroach/commits/dca415eddac0d659ae6d76b4e3dfdf4076adbd34):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-560-gdca415edda
Tests run against activerecord 7.0.3
7347 Total Tests Run
7333 tests passed
14 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
14 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::ConnectionAdapters::ConnectionHandlersShardingDbTest#test_same_shards_across_clusters (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
- #81088 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7697774?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7697774?buildTab=artifacts#/activerecord) on master @ [1a6e9f885baa124d5ff2996adb966ea15a1a9b2b](https://github.com/cockroachdb/cockroach/commits/1a6e9f885baa124d5ff2996adb966ea15a1a9b2b):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-648-g1a6e9f885b
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7709089?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7709089?buildTab=artifacts#/activerecord) on master @ [1a6e9f885baa124d5ff2996adb966ea15a1a9b2b](https://github.com/cockroachdb/cockroach/commits/1a6e9f885baa124d5ff2996adb966ea15a1a9b2b):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-648-g1a6e9f885b
Tests run against activerecord 7.0.3
7347 Total Tests Run
7329 tests passed
18 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
18 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_foreign_key_by_column_in_change_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_remove_reference_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_rename_reference_column_of_child_table (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::Migration::ForeignKeyChangeColumnWithPrefixTest#test_change_column_of_parent_table (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7742763?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7742763?buildTab=artifacts#/activerecord) on master @ [2b595c5214aead3e7d4f47f9a1dedce57dba6e7f](https://github.com/cockroachdb/cockroach/commits/2b595c5214aead3e7d4f47f9a1dedce57dba6e7f):


```
--- FAIL: MigrationTest#test_internal_metadata_stores_environment (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_virtual_column (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlIntegerTest#test_schema_properly_respects_bigint_ranges (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_with_if_exists_not_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_the_select_one_on_the_same_table (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_stored_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_add_foreign_key_with_on_update (unexpected)
--- FAIL: UpdateAllTest#test_update_bang_on_relation (unexpected)
--- FAIL: UpdateAllTest#test_update_all_with_joins_and_limit_and_order (unexpected)
--- FAIL: DefaultNumbersTest#test_default_decimal_number (unexpected)
--- FAIL: PostgresqlIntervalTest#test_column (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_symbol_column (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V4_2#test_polymorphic_add_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- FAIL: UpdateAllTest#test_update_counters_cares_about_optimistic_locking (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_singleton_has_many_through (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_schema_dumping_on_delete_and_on_update_options (unexpected)
--- FAIL: UpdateAllTest#test_update_on_relation_passing_active_record_object_is_not_permitted (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: PostgresqlIntervalTest#test_average_interval_type (unexpected)
--- FAIL: EagerSingularizationTest#test_eager_no_extra_singularization_has_many_through_has_many (unexpected)
--- FAIL: EagerSingularizationTest#test_eager_no_extra_singularization_has_many (unexpected)
--- FAIL: HasManyThroughAssociationsTest#test_ordered_has_many_through (unexpected)
--- FAIL: UpdateAllTest#test_update_all_doesnt_ignore_order (unexpected)
--- FAIL: EagerSingularizationTest#test_eager_no_extra_singularization_has_and_belongs_to_many (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_non_persisted_column (unexpected)
--- FAIL: UpdateAllTest#test_klass_level_touch_all (unexpected)
--- FAIL: UpdateAllTest#test_touch_all_cares_about_optimistic_locking (unexpected)
--- FAIL: UpdateAllTest#test_update_all_with_order_and_limit_and_offset_updates_subset_only (unexpected)
--- FAIL: UpdateAllTest#test_update_all_ignores_order_without_limit_from_association (unexpected)
--- FAIL: CommentTest#test_blank_columns_created_in_block (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: ActiveRecord::DatabaseTasksUtilsTask#test_raises_an_error_when_called_with_protected_environment (unexpected)
--- FAIL: EagerSingularizationTest#test_eager_no_extra_singularization_has_one (unexpected)
--- FAIL: LegacyPolymorphicReferenceIndexTest::V4_2#test_create_table_with_polymorphic_reference_uses_all_column_names_in_index (unexpected)
--- FAIL: UpdateAllTest#test_touch_all_updates_records_timestamps (unexpected)
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgresqlIntervalTest#test_interval_type_cast_from_invalid_string (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::ForeignKeyTest#test_remove_foreign_key_by_column (unexpected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: UpdateAllTest#test_update_all_with_group_by (unexpected)
--- FAIL: EagerSingularizationTest#test_eager_no_extra_singularization_has_many_through_belongs_to (unexpected)
--- FAIL: ActiveRecord::DatabaseTasksUtilsTask#test_raises_an_error_when_called_with_protected_environment_which_name_is_a_symbol (unexpected)
--- FAIL: PostgresqlVirtualColumnTest#test_change_table (unexpected)
--- FAIL: UpdateAllTest#test_klass_level_update_all (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7763932?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7763932?buildTab=artifacts#/activerecord) on master @ [7823e2f75474417a44f0d34f76c5f8914cb91d52](https://github.com/cockroachdb/cockroach/commits/7823e2f75474417a44f0d34f76c5f8914cb91d52):


```
test artifacts and logs in: /artifacts/activerecord/run_1
(test_impl.go:297).Fatalf: 
Tests run on Cockroach v23.1.0-alpha.00000000-749-g7823e2f754
Tests run against activerecord 7.0.3
7347 Total Tests Run
7334 tests passed
13 tests failed
26 tests skipped
5 tests ignored
0 tests passed unexpectedly
13 tests failed unexpectedly
0 tests expected failed but skipped
0 tests expected failed but not run
---
--- PASS: SerializedAttributeTest#test_unexpected_serialized_type (expected)
--- FAIL: PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tsrange (unexpected)
--- FAIL: PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date (unexpected)
--- FAIL: PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime (unexpected)
--- FAIL: PostgresqlRangeTest#test_timezone_array_awareness_tzrange (unexpected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol (unexpected)
--- PASS: SerializedAttributeTestWithYamlSafeLoad#test_unexpected_serialized_type (expected)
--- FAIL: ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7799005?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/7799005?buildTab=artifacts#/activerecord) on master @ [e7b15ebaed9c14668ade0a7827a5525aedef1ab0](https://github.com/cockroachdb/cockroach/commits/e7b15ebaed9c14668ade0a7827a5525aedef1ab0):


```
--- FAIL: TimestampTest#test_changing_parent_of_a_record_touches_both_new_and_old_polymorphic_parent_record_changes_with_other_class (unexpected)
--- FAIL: TouchTest#test_update (unexpected)
--- FAIL: ReservedWordTest#test_calculations_work_with_reserved_words (unexpected)
--- FAIL: HasManyAssociationsTest#test_transaction_when_deleting_persisted (unexpected)
--- FAIL: InverseBelongsToTests#test_child_instance_should_be_shared_with_replaced_via_accessor_parent (unexpected)
--- FAIL: TestHasOneAutosaveAssociationWhichItselfHasAutosaveAssociations#test_when_great-grandchild_changed_in_memory,_saving_parent_should_save_great-grandchild (unexpected)
--- FAIL: StrictLoadingTest#test_raises_on_lazy_loading_a_strict_loading_belongs_to_relation (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_one_should_count_instead_of_loading_association (unexpected)
--- FAIL: HasManyAssociationsTest#test_prevent_double_firing_the_before_save_callback_of_new_object_when_the_parent_association_saved_in_the_callback (unexpected)
--- FAIL: RelationTest#test_order_triggers_eager_loading_when_ordering_using_hash_syntax (unexpected)
--- FAIL: HasManyAssociationsTest#test_destroying_by_string_id (unexpected)
--- FAIL: TimestampTest#test_touching_update_at_attribute_as_symbol_updates_timestamp (unexpected)
--- FAIL: FoxyFixturesTest#test_populates_timestamp_columns (unexpected)
--- FAIL: HasManyAssociationsTest#test_deleting_before_save (unexpected)
--- FAIL: HasManyAssociationsTest#test_destroy_all_on_association_clears_scope (unexpected)
--- FAIL: RelationTest#test_finding_last_with_arel_order (unexpected)
--- FAIL: UniquenessValidationTest#test_validate_case_insensitive_uniqueness_with_special_sql_like_chars (unexpected)
--- FAIL: ActiveRecord::WhereChainTest#test_chaining_multiple (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_save_of_record_with_loaded_has_one_through (unexpected)
--- FAIL: HasManyAssociationsTest#test_delete_on_association_clears_scope (unexpected)
--- FAIL: FixturesTest#test_subsubdir_file_with_arbitrary_name (unexpected)
--- FAIL: BinaryTest#test_mixed_encoding (unexpected)
--- FAIL: HasManyAssociationsTest#test_get_ids_for_ordered_association (unexpected)
--- FAIL: ActiveRecord::PostgresqlConnectionTest#test_table_exists_logs_name (unexpected)
--- FAIL: TouchTest#test_create_turned_off (unexpected)
--- FAIL: FoxyFixturesTest#test_identifies_strings (unexpected)
--- FAIL: TypeTest#test_registering_a_new_type (unexpected)
--- FAIL: HotCompatibilityTest#test_update_after_remove_column (unexpected)
--- FAIL: InverseHasOneTests#test_parent_instance_should_be_shared_with_newly_built_child (unexpected)
--- FAIL: StrictLoadingTest#test_strict_loading_can_be_turned_off_on_an_association_in_a_model_with_strict_loading_on (unexpected)
--- FAIL: DefaultTextTest#test_default_texts_containing_single_quotes (unexpected)
--- FAIL: JsonAttributeTest#test_select_array_json_value (unexpected)
--- FAIL: ActiveRecord::ConnectionAdapters::ConnectionSwappingNestedTest#test_prevent_writes_can_be_changed_granularly (unexpected)
--- FAIL: FixturesTest#test_nonexistent_fixture_file (unexpected)
--- FAIL: TimestampTest#test_timestamp_column_values_are_present_in_the_callbacks (unexpected)
--- FAIL: RelationTest#test_to_yaml (unexpected)
--- FAIL: RelationTest#test_joins_with_select_custom_attribute (unexpected)
--- FAIL: PostgresqlInfinityTest#test_where_clause_with_infinite_range_on_a_datetime_column (unexpected)
--- FAIL: StrictLoadingTest#test_strict_loading_with_reflection_is_ignored_in_validation_context (unexpected)
--- FAIL: HasOneThroughAssociationsTest#test_has_one_through_do_not_cache_association_reader_if_the_though_method_has_default_scopes (unexpected)
--- FAIL: HasManyAssociationsTest#test_calling_empty_on_an_association_that_has_not_been_loaded_performs_a_query (unexpected)
--- FAIL: RelationTest#test_size_with_limit (unexpected)
--- FAIL: ActiveRecord::Coders::YAMLColumnTestWithSafeLoad#test_returns_string_unless_starts_with_dash (unexpected)
--- FAIL: RelationTest#test_select_takes_a_variable_list_of_args (unexpected)
--- FAIL: ActiveRecord::WhereChainTest#test_rewhere_with_alias_condition (unexpected)
--- FAIL: StrictLoadingTest#test_raises_on_unloaded_relation_methods_if_strict_loading (unexpected)
--- FAIL: LengthValidationTest#test_validates_size_of_association_using_within (unexpected)
--- FAIL: TransactionsWithTransactionalFixturesTest#test_no_automatic_savepoint_for_inner_transaction (unexpected)
For a full summary look at the activerecord artifacts
An updated blocklist (activeRecordBlockList23_1) is available in the artifacts' activerecord log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #89367 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.2.0]
- #87580 roachtest: activerecord failed [C-test-failure O-roachtest O-robot branch-release-22.2]
- #82805 roachtest: activerecord failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/roachtest/tests/activerecord.go
+++ b/pkg/cmd/roachtest/tests/activerecord.go
@@ -164,15 +164,9 @@ func registerActiveRecord(r registry.Registry) {
 			t.Fatal(err)
 		}
 
-		blocklistName, expectedFailures, ignorelistName, ignorelist := activeRecordBlocklists.getLists(version)
-		if expectedFailures == nil {
-			t.Fatalf("No activerecord blocklist defined for cockroach version %s", version)
-		}
-		status := fmt.Sprintf("Running cockroach version %s, using blocklist %s", version, blocklistName)
-		if ignorelist != nil {
-			status = fmt.Sprintf("Running cockroach version %s, using blocklist %s, using ignorelist %s",
-				version, blocklistName, ignorelistName)
-		}
+		blocklistName, ignorelistName := "activeRecordBlocklist", "activeRecordIgnoreList"
+		status := fmt.Sprintf("Running cockroach version %s, using blocklist %s, using ignorelist %s",
+			version, blocklistName, ignorelistName)
 		t.L().Printf("%s", status)
 
 		t.Status("running activerecord test suite")
@@ -206,8 +200,8 @@ func registerActiveRecord(r registry.Registry) {
 			skipped := result == "S"
 			results.allTests = append(results.allTests, test)
 
-			ignoredIssue, expectedIgnored := ignorelist[test]
-			issue, expectedFailure := expectedFailures[test]
+			ignoredIssue, expectedIgnored := activeRecordIgnoreList[test]
+			issue, expectedFailure := activeRecordBlocklist[test]
 			switch {
 			case expectedIgnored:
 				results.results[test] = fmt.Sprintf("--- SKIP: %s due to %s (expected)", test, ignoredIssue)
@@ -241,7 +235,7 @@ func registerActiveRecord(r registry.Registry) {
 		}
 
 		results.summarizeAll(
-			t, "activerecord" /* ormName */, blocklistName, expectedFailures, version, supportedRailsVersion,
+			t, "activerecord" /* ormName */, blocklistName, activeRecordBlocklist, version, supportedRailsVersion,
 		)
 	}
 

--- a/pkg/cmd/roachtest/tests/activerecord_blocklist.go
+++ b/pkg/cmd/roachtest/tests/activerecord_blocklist.go
@@ -10,15 +10,6 @@
 
 package tests
 
-var activeRecordBlocklists = blocklistsForVersion{
-	{"v20.2", "activeRecordBlockList20_2", activeRecordBlockList20_2, "activeRecordIgnoreList20_2", activeRecordIgnoreList20_2},
-	{"v21.1", "activeRecordBlockList21_1", activeRecordBlockList21_1, "activeRecordIgnoreList21_1", activeRecordIgnoreList21_1},
-	{"v21.2", "activeRecordBlockList21_2", activeRecordBlockList21_2, "activeRecordIgnoreList21_2", activeRecordIgnoreList21_2},
-	{"v22.1", "activeRecordBlockList22_1", activeRecordBlockList22_1, "activeRecordIgnoreList22_1", activeRecordIgnoreList22_1},
-	{"v22.2", "activeRecordBlockList22_2", activeRecordBlockList22_2, "activeRecordIgnoreList22_2", activeRecordIgnoreList22_2},
-	{"v23.1", "activeRecordBlockList23_1", activeRecordBlockList23_1, "activeRecordIgnoreList23_1", activeRecordIgnoreList23_1},
-}
-
 // These are lists of known activerecord test errors and failures.
 // When the activerecord test suite is run, the results are compared to this list.
 // Any passed test that is not on this list is reported as PASS - expected
@@ -30,51 +21,76 @@ var activeRecordBlocklists = blocklistsForVersion{
 // Please keep these lists alphabetized for easy diffing.
 // After a failed run, an updated version of this blocklist should be available
 // in the test log.
-var activeRecordBlockList23_1 = blocklist{}
-
-var activeRecordBlockList22_2 = blocklist{}
-
-var activeRecordBlockList22_1 = blocklist{}
-
-var activeRecordBlockList21_2 = blocklist{}
-
-var activeRecordBlockList21_1 = blocklist{}
-
-var activeRecordBlockList20_2 = blocklist{}
-
-var activeRecordIgnoreList23_1 = activeRecordIgnoreList22_2
-
-var activeRecordIgnoreList22_2 = activeRecordIgnoreList22_1
-
-var activeRecordIgnoreList22_1 = blocklist{
-	"CockroachDB::PostgresqlIntervalTest#test_interval_type":               "flaky",
-	"ConcurrentTransactionTest#test_transaction_isolation__read_committed": "flaky - https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/237",
-	"FixturesTest#test_create_fixtures":                                    "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_books_fixtures":                       "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_parrots_fixtures":                     "flaky - FK constraint violated sometimes when loading all fixture data",
-	"PostgresqlIntervalTest#test_interval_type":                            "flaky",
+var activeRecordBlocklist = blocklist{
+	"ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_string_to_date":                                               "49329",
+	"ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_array":                                              "49329",
+	"ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol":                                             "49329",
+	"ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime":                              "49329",
+	"ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_datetime_with_timestamptz_as_default":  "49329",
+	"ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_using_timestamp_with_timestamptz_as_default": "49329",
+	"ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz_as_default":                 "49329",
+	"ActiveRecord::CockroachDB::Migration::PGChangeSchemaTest#test_change_type_with_symbol_with_timestamptz":                            "49329",
+	"PostgresqlRangeTest#test_timezone_array_awareness_tsrange":                                                                         "27791",
+	"PostgresqlRangeTest#test_timezone_array_awareness_tzrange":                                                                         "27791",
+	// TODO(yang): investigate cause further
+	"PostgreSQLReferentialIntegrityTest#test_all_foreign_keys_valid_having_foreign_keys_in_multiple_schemas":                             "unknown",
+	"PostgresqlTimestampWithAwareTypesTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_time_zone_set":               "expected ActiveSupport::TimeWithZone, not Time",
+	"PostgresqlTimestampWithTimeZoneTest#test_timestamp_with_zone_values_with_rails_time_zone_support_and_timestamptz_and_time_zone_set": "expected ActiveSupport::TimeWithZone, not Time",
 }
 
-var activeRecordIgnoreList21_2 = blocklist{
-	"ActiveRecord::CockroachDB::UnloggedTablesTest#test_gracefully_handles_temporary_tables": "modified to pass on 20.2",
-	"CockroachDB::PostgresqlIntervalTest#test_interval_type":                                 "flaky",
-	"ConcurrentTransactionTest#test_transaction_isolation__read_committed":                   "flaky - https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/237",
-	"FixturesTest#test_create_fixtures":                                                      "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_books_fixtures":                                         "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_parrots_fixtures":                                       "flaky - FK constraint violated sometimes when loading all fixture data",
-}
-
-var activeRecordIgnoreList21_1 = blocklist{
-	"ActiveRecord::CockroachDB::UnloggedTablesTest#test_gracefully_handles_temporary_tables": "modified to pass on 20.2",
-	"CockroachDB::PostgresqlIntervalTest#test_interval_type":                                 "flaky",
-	"FixturesTest#test_create_fixtures":                                                      "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_books_fixtures":                                         "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_parrots_fixtures":                                       "flaky - FK constraint violated sometimes when loading all fixture data",
-}
-
-var activeRecordIgnoreList20_2 = blocklist{
-	"ActiveRecord::CockroachDB::UnloggedTablesTest#test_gracefully_handles_temporary_tables": "modified to pass on 20.2",
-	"FixturesTest#test_create_fixtures":                "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_books_fixtures":   "flaky - FK constraint violated sometimes when loading all fixture data",
-	"IgnoreFixturesTest#test_ignores_parrots_fixtures": "flaky - FK constraint violated sometimes when loading all fixture data",
+var activeRecordIgnoreList = blocklist{
+	"ActiveRecord::Coders::YAMLColumnTestWithSafeLoad#test_returns_string_unless_starts_with_dash":                                                             "flaky - sometimes attempts to call a method on nil",
+	"ActiveRecord::ConnectionAdapters::ConnectionSwappingNestedTest#test_prevent_writes_can_be_changed_granularly":                                             "flaky - sometimes attempts to call a method on nil",
+	"ActiveRecord::PostgresqlConnectionTest#test_table_exists_logs_name":                                                                                       "flaky - sometimes attempts to call a method on nil",
+	"ActiveRecord::WhereChainTest#test_chaining_multiple":                                                                                                      "flaky - sometimes complains that a relation does not exist",
+	"BinaryTest#test_mixed_encoding":                                                                                                                           "flaky - sometimes complains that a relation does not exist",
+	"CockroachDB::FixturesTest#test_bulk_insert":                                                                                                               "flaky",
+	"CockroachDB::PostgresqlIntervalTest#test_interval_type":                                                                                                   "flaky",
+	"ConcurrentTransactionTest#test_transaction_isolation__read_committed":                                                                                     "flaky - https://github.com/cockroachdb/activerecord-cockroachdb-adapter/issues/237",
+	"DefaultTextTest#test_default_texts_containing_single_quotes":                                                                                              "flaky - sometimes complains that a relation does not exist",
+	"FixturesTest#test_create_fixtures":                                                                                                                        "flaky - FK constraint violated sometimes when loading all fixture data",
+	"FixturesTest#test_nonexistent_fixture_file":                                                                                                               "flaky - sometimes complains that a relation does not exist",
+	"FixturesTest#test_subsubdir_file_with_arbitrary_name":                                                                                                     "flaky - sometimes complains that a relation does not exist",
+	"FoxyFixturesTest#test_identifies_strings":                                                                                                                 "flaky - sometimes complains that a relation does not exist",
+	"FoxyFixturesTest#test_populates_timestamp_column":                                                                                                         "flaky - sometimes complains that a relation does not exist",
+	"HasManyAssociationsTest#test_calling_empty_on_an_association_that_has_not_been_loaded_performs_a_query":                                                   "flaky - sometimes complains that a relation does not exist",
+	"HasManyAssociationsTest#test_calling_one_should_count_instead_of_loading_association":                                                                     "flaky - sometimes complains that a relation does not exist",
+	"HasManyAssociationsTest#test_delete_on_association_clears_scope":                                                                                          "flaky - sometimes complains that a relation does not exist",
+	"HasManyAssociationsTest#test_deleting_before_save":                                                                                                        "flaky - sometimes complains that a relation does not exist",
+	"HasManyAssociationsTest#test_destroy_all_on_association_clears_scope":                                                                                     "flaky - sometimes complains that a relation does not exist",
+	"HasManyAssociationsTest#test_destroying_by_string_id":                                                                                                     "flaky - sometimes complains that a relation does not exist",
+	"HasManyAssociationsTest#test_get_ids_for_ordered_association":                                                                                             "flaky - sometimes complains that a relation does not exist",
+	"HasOneThroughAssociationsTest#test_has_one_through_do_not_cache_association_reader_if_the_though_method_has_default_scopes":                               "flaky - sometimes complains that a relation does not exist",
+	"HasOneThroughAssociationsTest#test_save_of_record_with_loaded_has_one_through":                                                                            "flaky - sometimes complains that a relation does not exist",
+	"HasManyAssociationsTest#test_prevent_double_firing_the_before_save_callback_of_new_object_when_the_parent_association_saved_in_the_callback":              "flaky - sometimes complains that a relation does not exist",
+	"HasManyAssociationsTest#test_transaction_when_deleting_persisted":                                                                                         "flaky - sometimes complains that a relation does not exist",
+	"HotCompatibilityTest#test_update_after_remove_column":                                                                                                     "flaky - sometimes attempts to call a method on nil",
+	"InverseBelongsToTests#test_child_instance_should_be_shared_with_replaced_via_accessor_parent":                                                             "flaky - sometimes complains that a relation does not exist",
+	"InverseHasOneTests#test_parent_instance_should_be_shared_with_newly_built_child":                                                                          "flaky - sometimes complains that a relation does not exist",
+	"IgnoreFixturesTest#test_ignores_books_fixtures":                                                                                                           "flaky - FK constraint violated sometimes when loading all fixture data",
+	"IgnoreFixturesTest#test_ignores_parrots_fixtures":                                                                                                         "flaky - FK constraint violated sometimes when loading all fixture data",
+	"JsonAttributeTest#test_select_array_json_value":                                                                                                           "flaky - sometimes attempts to call a method on nil",
+	"LengthValidationTest#test_validates_size_of_association_using_within":                                                                                     "flaky - sometimes complains that a relation does not exist",
+	"PostgresqlInfinityTest#test_where_clause_with_infinite_range_on_a_datetime_column":                                                                        "flaky - sometimes complains that a relation does not exist",
+	"PostgresqlIntervalTest#test_interval_type":                                                                                                                "flaky",
+	"RelationTest#test_finding_last_with_arel_order":                                                                                                           "flaky - sometimes complains that a relation does not exist",
+	"RelationTest#test_joins_with_select_custom_attribute":                                                                                                     "flaky - sometimes complains that a relation does not exist",
+	"RelationTest#test_order_triggers_eager_loading_when_ordering_using_hash_syntax":                                                                           "flaky - sometimes complains that a relation does not exist",
+	"RelationTest#test_select_takes_a_variable_list_of_args":                                                                                                   "flaky - sometimes complains that a relation does not exist",
+	"RelationTest#test_size_with_limit":                                                                                                                        "flaky - sometimes complains that a relation does not exist",
+	"RelationTest#test_to_yaml":                                                                                                                                "flaky - sometimes complains that a relation does not exist",
+	"ReservedWordTest#test_calculations_work_with_reserved_words":                                                                                              "flaky - sometimes complains that a relation does not exist",
+	"StrictLoadingTest#test_raises_on_lazy_loading_a_strict_loading_belongs_to_relation":                                                                       "flaky - sometimes complains that a relation does not exist",
+	"StrictLoadingTest#test_raises_on_unloaded_relation_methods_if_strict_loading":                                                                             "flaky - sometimes complains that a relation does not exist",
+	"StrictLoadingTest#test_strict_loading_can_be_turned_off_on_an_association_in_a_model_with_strict_loading_on":                                              "flaky - sometimes complains that a relation does not exist",
+	"StrictLoadingTest#test_strict_loading_with_reflection_is_ignored_in_validation_context":                                                                   "flaky - sometimes complains that a relation does not exist",
+	"TestHasOneAutosaveAssociationWhichItselfHasAutosaveAssociations#test_when_great-grandchild_changed_in_memory,_saving_parent_should_save_great-grandchild": "flaky - sometimes attempts to call a method on nil",
+	"TimestampTest#test_changing_parent_of_a_record_touches_both_new_and_old_polymorphic_parent_record_changes_with_other_class":                               "flaky - sometimes complains that a relation does not exist",
+	"TimestampTest#test_timestamp_column_values_are_present_in_the_callbacks":                                                                                  "flaky - sometimes complains that a relation does not exist",
+	"TimestampTest#test_touching_update_at_attribute_as_symbol_updates_timestamp":                                                                              "flaky - sometimes complains that a relation does not exist",
+	"TouchTest#test_create_turned_off":                                                                                                                         "flaky - sometimes complains that a relation does not exist",
+	"TouchTest#test_update":                                                                                                                                    "flaky - sometimes complains that a relation does not exist",
+	"TransactionsWithTransactionalFixturesTest#test_no_automatic_savepoint_for_inner_transaction":                                                              "flaky - sometimes attempts to call a method on nil",
+	"TypeTest#test_registering_a_new_type":                                                                                                                     "flaky - sometimes attempts to call a method on nil",
+	"UniquenessValidationTest#test_validate_case_insensitive_uniqueness_with_special_sql_like_chars":                                                           "flaky - sometimes complains that a relation does not exist",
 }

--- a/pkg/cmd/roachtest/tests/blocklist_test.go
+++ b/pkg/cmd/roachtest/tests/blocklist_test.go
@@ -41,7 +41,7 @@ func TestBlocklists(t *testing.T) {
 		"libpq":        libPQBlocklist20_2,
 		"gopg":         gopgBlockList20_2,
 		"pgx":          pgxBlocklist20_2,
-		"activerecord": activeRecordBlockList20_2,
+		"activerecord": activeRecordBlocklist,
 	}
 	type reasonCount struct {
 		reason string

